<html><head><meta content="text/html; charset=UTF-8" http-equiv="content-type"><style type="text/css">@import url('https://themes.googleusercontent.com/fonts/css?kit=384rsQkuuLKrqu_E2x74BeIvZw9xjkfKFpvLW8okg10');ol.lst-kix_bano2ezcy3fm-8.start{counter-reset:lst-ctn-kix_bano2ezcy3fm-8 0}.lst-kix_cxfq5ti6eb8z-6>li:before{content:"" counter(lst-ctn-kix_cxfq5ti6eb8z-6,decimal) ". "}.lst-kix_cxfq5ti6eb8z-4>li:before{content:"" counter(lst-ctn-kix_cxfq5ti6eb8z-4,lower-latin) ". "}.lst-kix_cxfq5ti6eb8z-8>li:before{content:"" counter(lst-ctn-kix_cxfq5ti6eb8z-8,lower-roman) ". "}.lst-kix_cxfq5ti6eb8z-3>li:before{content:"" counter(lst-ctn-kix_cxfq5ti6eb8z-3,decimal) ". "}.lst-kix_cxfq5ti6eb8z-7>li:before{content:"" counter(lst-ctn-kix_cxfq5ti6eb8z-7,lower-latin) ". "}.lst-kix_cxfq5ti6eb8z-2>li:before{content:"" counter(lst-ctn-kix_cxfq5ti6eb8z-2,lower-roman) ". "}.lst-kix_wd70178zyj44-5>li{counter-increment:lst-ctn-kix_wd70178zyj44-5}ol.lst-kix_6zwgo5b1uhw7-0.start{counter-reset:lst-ctn-kix_6zwgo5b1uhw7-0 0}.lst-kix_cxfq5ti6eb8z-1>li:before{content:"" counter(lst-ctn-kix_cxfq5ti6eb8z-1,lower-latin) ". "}ol.lst-kix_gen1172f8ewp-1.start{counter-reset:lst-ctn-kix_gen1172f8ewp-1 0}.lst-kix_cxfq5ti6eb8z-0>li:before{content:"" counter(lst-ctn-kix_cxfq5ti6eb8z-0,decimal) ". "}ol.lst-kix_wd70178zyj44-8{list-style-type:none}ol.lst-kix_gen1172f8ewp-7.start{counter-reset:lst-ctn-kix_gen1172f8ewp-7 0}.lst-kix_bano2ezcy3fm-1>li{counter-increment:lst-ctn-kix_bano2ezcy3fm-1}ol.lst-kix_avu7jq5xuo82-1.start{counter-reset:lst-ctn-kix_avu7jq5xuo82-1 0}ol.lst-kix_cxfq5ti6eb8z-4.start{counter-reset:lst-ctn-kix_cxfq5ti6eb8z-4 0}ol.lst-kix_wd70178zyj44-2.start{counter-reset:lst-ctn-kix_wd70178zyj44-2 0}ol.lst-kix_bano2ezcy3fm-2.start{counter-reset:lst-ctn-kix_bano2ezcy3fm-2 0}ol.lst-kix_kg9nuuktziog-5.start{counter-reset:lst-ctn-kix_kg9nuuktziog-5 0}.lst-kix_cxfq5ti6eb8z-0>li{counter-increment:lst-ctn-kix_cxfq5ti6eb8z-0}.lst-kix_u0t4yt1grtwe-8>li{counter-increment:lst-ctn-kix_u0t4yt1grtwe-8}.lst-kix_bano2ezcy3fm-8>li:before{content:"" counter(lst-ctn-kix_bano2ezcy3fm-8,lower-roman) ". "}ol.lst-kix_u0t4yt1grtwe-1.start{counter-reset:lst-ctn-kix_u0t4yt1grtwe-1 0}.lst-kix_avu7jq5xuo82-5>li{counter-increment:lst-ctn-kix_avu7jq5xuo82-5}ol.lst-kix_fplirl6b8ae4-3.start{counter-reset:lst-ctn-kix_fplirl6b8ae4-3 0}ol.lst-kix_wd70178zyj44-8.start{counter-reset:lst-ctn-kix_wd70178zyj44-8 0}.lst-kix_bano2ezcy3fm-2>li:before{content:"" counter(lst-ctn-kix_bano2ezcy3fm-2,lower-roman) ". "}.lst-kix_bano2ezcy3fm-3>li:before{content:"" counter(lst-ctn-kix_bano2ezcy3fm-3,decimal) ". "}ol.lst-kix_avu7jq5xuo82-7.start{counter-reset:lst-ctn-kix_avu7jq5xuo82-7 0}.lst-kix_bano2ezcy3fm-4>li:before{content:"" counter(lst-ctn-kix_bano2ezcy3fm-4,lower-latin) ". "}.lst-kix_gen1172f8ewp-0>li{counter-increment:lst-ctn-kix_gen1172f8ewp-0}.lst-kix_wljc02q05kpj-0>li{counter-increment:lst-ctn-kix_wljc02q05kpj-0}.lst-kix_wd70178zyj44-7>li{counter-increment:lst-ctn-kix_wd70178zyj44-7}.lst-kix_bano2ezcy3fm-5>li:before{content:"" counter(lst-ctn-kix_bano2ezcy3fm-5,lower-roman) ". "}.lst-kix_bano2ezcy3fm-7>li:before{content:"" counter(lst-ctn-kix_bano2ezcy3fm-7,lower-latin) ". "}ol.lst-kix_wljc02q05kpj-7.start{counter-reset:lst-ctn-kix_wljc02q05kpj-7 0}.lst-kix_cxfq5ti6eb8z-5>li:before{content:"" counter(lst-ctn-kix_cxfq5ti6eb8z-5,lower-roman) ". "}.lst-kix_bano2ezcy3fm-6>li:before{content:"" counter(lst-ctn-kix_bano2ezcy3fm-6,decimal) ". "}.lst-kix_6zwgo5b1uhw7-5>li{counter-increment:lst-ctn-kix_6zwgo5b1uhw7-5}.lst-kix_avu7jq5xuo82-1>li{counter-increment:lst-ctn-kix_avu7jq5xuo82-1}ol.lst-kix_fplirl6b8ae4-4.start{counter-reset:lst-ctn-kix_fplirl6b8ae4-4 0}ol.lst-kix_avu7jq5xuo82-1{list-style-type:none}ol.lst-kix_avu7jq5xuo82-0{list-style-type:none}ol.lst-kix_avu7jq5xuo82-7{list-style-type:none}ol.lst-kix_avu7jq5xuo82-6{list-style-type:none}.lst-kix_kg9nuuktziog-6>li{counter-increment:lst-ctn-kix_kg9nuuktziog-6}.lst-kix_bano2ezcy3fm-1>li:before{content:"" counter(lst-ctn-kix_bano2ezcy3fm-1,lower-latin) ". "}ol.lst-kix_gen1172f8ewp-2.start{counter-reset:lst-ctn-kix_gen1172f8ewp-2 0}ol.lst-kix_avu7jq5xuo82-8{list-style-type:none}ol.lst-kix_gen1172f8ewp-8{list-style-type:none}.lst-kix_wljc02q05kpj-2>li{counter-increment:lst-ctn-kix_wljc02q05kpj-2}ol.lst-kix_avu7jq5xuo82-3{list-style-type:none}ol.lst-kix_avu7jq5xuo82-2{list-style-type:none}ol.lst-kix_gen1172f8ewp-6{list-style-type:none}.lst-kix_fplirl6b8ae4-3>li{counter-increment:lst-ctn-kix_fplirl6b8ae4-3}ol.lst-kix_avu7jq5xuo82-5{list-style-type:none}ol.lst-kix_u0t4yt1grtwe-7.start{counter-reset:lst-ctn-kix_u0t4yt1grtwe-7 0}.lst-kix_bano2ezcy3fm-0>li:before{content:"" counter(lst-ctn-kix_bano2ezcy3fm-0,decimal) ". "}ol.lst-kix_gen1172f8ewp-7{list-style-type:none}ol.lst-kix_avu7jq5xuo82-4{list-style-type:none}.lst-kix_cxfq5ti6eb8z-7>li{counter-increment:lst-ctn-kix_cxfq5ti6eb8z-7}.lst-kix_avu7jq5xuo82-7>li{counter-increment:lst-ctn-kix_avu7jq5xuo82-7}ol.lst-kix_gen1172f8ewp-4{list-style-type:none}.lst-kix_fplirl6b8ae4-6>li{counter-increment:lst-ctn-kix_fplirl6b8ae4-6}ol.lst-kix_wd70178zyj44-6{list-style-type:none}ol.lst-kix_gen1172f8ewp-5{list-style-type:none}ol.lst-kix_wd70178zyj44-7{list-style-type:none}ol.lst-kix_gen1172f8ewp-2{list-style-type:none}ol.lst-kix_wd70178zyj44-4{list-style-type:none}ol.lst-kix_gen1172f8ewp-3{list-style-type:none}.lst-kix_gen1172f8ewp-5>li{counter-increment:lst-ctn-kix_gen1172f8ewp-5}ol.lst-kix_wd70178zyj44-5{list-style-type:none}ol.lst-kix_gen1172f8ewp-0{list-style-type:none}ol.lst-kix_wd70178zyj44-2{list-style-type:none}ol.lst-kix_gen1172f8ewp-1{list-style-type:none}ol.lst-kix_wd70178zyj44-3{list-style-type:none}ol.lst-kix_u0t4yt1grtwe-6.start{counter-reset:lst-ctn-kix_u0t4yt1grtwe-6 0}ol.lst-kix_wd70178zyj44-0{list-style-type:none}ol.lst-kix_avu7jq5xuo82-8.start{counter-reset:lst-ctn-kix_avu7jq5xuo82-8 0}.lst-kix_gen1172f8ewp-2>li{counter-increment:lst-ctn-kix_gen1172f8ewp-2}ol.lst-kix_wd70178zyj44-1{list-style-type:none}.lst-kix_bano2ezcy3fm-3>li{counter-increment:lst-ctn-kix_bano2ezcy3fm-3}ol.lst-kix_u0t4yt1grtwe-0.start{counter-reset:lst-ctn-kix_u0t4yt1grtwe-0 0}ol.lst-kix_wd70178zyj44-3.start{counter-reset:lst-ctn-kix_wd70178zyj44-3 0}ol.lst-kix_avu7jq5xuo82-2.start{counter-reset:lst-ctn-kix_avu7jq5xuo82-2 0}ol.lst-kix_cxfq5ti6eb8z-5.start{counter-reset:lst-ctn-kix_cxfq5ti6eb8z-5 0}ol.lst-kix_bano2ezcy3fm-7.start{counter-reset:lst-ctn-kix_bano2ezcy3fm-7 0}.lst-kix_u0t4yt1grtwe-6>li{counter-increment:lst-ctn-kix_u0t4yt1grtwe-6}ol.lst-kix_6zwgo5b1uhw7-7.start{counter-reset:lst-ctn-kix_6zwgo5b1uhw7-7 0}.lst-kix_wljc02q05kpj-3>li:before{content:"" counter(lst-ctn-kix_wljc02q05kpj-3,decimal) ". "}.lst-kix_wd70178zyj44-1>li:before{content:"" counter(lst-ctn-kix_wd70178zyj44-1,upper-latin) ". "}.lst-kix_pb7y9mdhka33-4>li:before{content:"-  "}.lst-kix_pb7y9mdhka33-6>li:before{content:"-  "}ol.lst-kix_fplirl6b8ae4-2.start{counter-reset:lst-ctn-kix_fplirl6b8ae4-2 0}.lst-kix_wljc02q05kpj-7>li:before{content:"" counter(lst-ctn-kix_wljc02q05kpj-7,lower-latin) ". "}.lst-kix_wd70178zyj44-3>li:before{content:"" counter(lst-ctn-kix_wd70178zyj44-3,lower-latin) ") "}.lst-kix_wd70178zyj44-5>li:before{content:"(" counter(lst-ctn-kix_wd70178zyj44-5,lower-latin) ") "}.lst-kix_wljc02q05kpj-5>li:before{content:"" counter(lst-ctn-kix_wljc02q05kpj-5,lower-roman) ". "}ol.lst-kix_6zwgo5b1uhw7-4.start{counter-reset:lst-ctn-kix_6zwgo5b1uhw7-4 0}ol.lst-kix_kg9nuuktziog-1.start{counter-reset:lst-ctn-kix_kg9nuuktziog-1 0}.lst-kix_kg9nuuktziog-3>li{counter-increment:lst-ctn-kix_kg9nuuktziog-3}ol.lst-kix_wljc02q05kpj-3.start{counter-reset:lst-ctn-kix_wljc02q05kpj-3 0}.lst-kix_pb7y9mdhka33-0>li:before{content:"-  "}.lst-kix_pb7y9mdhka33-2>li:before{content:"-  "}.lst-kix_wljc02q05kpj-1>li:before{content:"" counter(lst-ctn-kix_wljc02q05kpj-1,lower-latin) ". "}.lst-kix_fplirl6b8ae4-7>li:before{content:"" counter(lst-ctn-kix_fplirl6b8ae4-7,lower-latin) ". "}ol.lst-kix_avu7jq5xuo82-3.start{counter-reset:lst-ctn-kix_avu7jq5xuo82-3 0}.lst-kix_cxfq5ti6eb8z-3>li{counter-increment:lst-ctn-kix_cxfq5ti6eb8z-3}.lst-kix_fplirl6b8ae4-5>li:before{content:"" counter(lst-ctn-kix_fplirl6b8ae4-5,lower-roman) ". "}.lst-kix_bano2ezcy3fm-5>li{counter-increment:lst-ctn-kix_bano2ezcy3fm-5}ol.lst-kix_gen1172f8ewp-5.start{counter-reset:lst-ctn-kix_gen1172f8ewp-5 0}.lst-kix_6zwgo5b1uhw7-3>li{counter-increment:lst-ctn-kix_6zwgo5b1uhw7-3}.lst-kix_avu7jq5xuo82-8>li{counter-increment:lst-ctn-kix_avu7jq5xuo82-8}.lst-kix_wljc02q05kpj-6>li{counter-increment:lst-ctn-kix_wljc02q05kpj-6}.lst-kix_fplirl6b8ae4-3>li:before{content:"" counter(lst-ctn-kix_fplirl6b8ae4-3,decimal) ". "}.lst-kix_kg9nuuktziog-1>li{counter-increment:lst-ctn-kix_kg9nuuktziog-1}ol.lst-kix_wljc02q05kpj-0.start{counter-reset:lst-ctn-kix_wljc02q05kpj-0 0}ol.lst-kix_wd70178zyj44-4.start{counter-reset:lst-ctn-kix_wd70178zyj44-4 0}.lst-kix_wd70178zyj44-7>li:before{content:"(" counter(lst-ctn-kix_wd70178zyj44-7,lower-latin) ") "}.lst-kix_fplirl6b8ae4-1>li:before{content:"" counter(lst-ctn-kix_fplirl6b8ae4-1,lower-latin) ". "}.lst-kix_pb7y9mdhka33-8>li:before{content:"-  "}ol.lst-kix_u0t4yt1grtwe-8.start{counter-reset:lst-ctn-kix_u0t4yt1grtwe-8 0}.lst-kix_gen1172f8ewp-6>li{counter-increment:lst-ctn-kix_gen1172f8ewp-6}ol.lst-kix_gen1172f8ewp-8.start{counter-reset:lst-ctn-kix_gen1172f8ewp-8 0}ol.lst-kix_cxfq5ti6eb8z-0.start{counter-reset:lst-ctn-kix_cxfq5ti6eb8z-0 0}.lst-kix_6zwgo5b1uhw7-6>li:before{content:"" counter(lst-ctn-kix_6zwgo5b1uhw7-6,decimal) ". "}.lst-kix_u0t4yt1grtwe-6>li:before{content:"" counter(lst-ctn-kix_u0t4yt1grtwe-6,decimal) ". "}ol.lst-kix_wd70178zyj44-7.start{counter-reset:lst-ctn-kix_wd70178zyj44-7 0}ol.lst-kix_cxfq5ti6eb8z-3.start{counter-reset:lst-ctn-kix_cxfq5ti6eb8z-3 0}.lst-kix_fplirl6b8ae4-8>li{counter-increment:lst-ctn-kix_fplirl6b8ae4-8}ol.lst-kix_wljc02q05kpj-1.start{counter-reset:lst-ctn-kix_wljc02q05kpj-1 0}.lst-kix_wljc02q05kpj-7>li{counter-increment:lst-ctn-kix_wljc02q05kpj-7}.lst-kix_u0t4yt1grtwe-2>li:before{content:"" counter(lst-ctn-kix_u0t4yt1grtwe-2,lower-roman) ". "}.lst-kix_6zwgo5b1uhw7-2>li:before{content:"" counter(lst-ctn-kix_6zwgo5b1uhw7-2,lower-roman) ". "}.lst-kix_6zwgo5b1uhw7-8>li:before{content:"" counter(lst-ctn-kix_6zwgo5b1uhw7-8,lower-roman) ". "}.lst-kix_u0t4yt1grtwe-0>li:before{content:"" counter(lst-ctn-kix_u0t4yt1grtwe-0,decimal) ". "}.lst-kix_u0t4yt1grtwe-8>li:before{content:"" counter(lst-ctn-kix_u0t4yt1grtwe-8,lower-roman) ". "}ol.lst-kix_6zwgo5b1uhw7-5.start{counter-reset:lst-ctn-kix_6zwgo5b1uhw7-5 0}.lst-kix_wljc02q05kpj-1>li{counter-increment:lst-ctn-kix_wljc02q05kpj-1}ol.lst-kix_gen1172f8ewp-6.start{counter-reset:lst-ctn-kix_gen1172f8ewp-6 0}.lst-kix_gen1172f8ewp-7>li{counter-increment:lst-ctn-kix_gen1172f8ewp-7}ol.lst-kix_6zwgo5b1uhw7-7{list-style-type:none}.lst-kix_gen1172f8ewp-5>li:before{content:"(" counter(lst-ctn-kix_gen1172f8ewp-5,lower-latin) ") "}.lst-kix_gen1172f8ewp-7>li:before{content:"(" counter(lst-ctn-kix_gen1172f8ewp-7,lower-latin) ") "}ol.lst-kix_6zwgo5b1uhw7-8{list-style-type:none}.lst-kix_gen1172f8ewp-1>li{counter-increment:lst-ctn-kix_gen1172f8ewp-1}ol.lst-kix_6zwgo5b1uhw7-5{list-style-type:none}ol.lst-kix_fplirl6b8ae4-0.start{counter-reset:lst-ctn-kix_fplirl6b8ae4-0 0}ol.lst-kix_6zwgo5b1uhw7-6{list-style-type:none}ol.lst-kix_6zwgo5b1uhw7-3{list-style-type:none}.lst-kix_fplirl6b8ae4-7>li{counter-increment:lst-ctn-kix_fplirl6b8ae4-7}ol.lst-kix_6zwgo5b1uhw7-4{list-style-type:none}.lst-kix_bano2ezcy3fm-4>li{counter-increment:lst-ctn-kix_bano2ezcy3fm-4}ol.lst-kix_6zwgo5b1uhw7-1{list-style-type:none}.lst-kix_u0t4yt1grtwe-4>li{counter-increment:lst-ctn-kix_u0t4yt1grtwe-4}ol.lst-kix_6zwgo5b1uhw7-2{list-style-type:none}ol.lst-kix_6zwgo5b1uhw7-0{list-style-type:none}.lst-kix_6zwgo5b1uhw7-4>li:before{content:"" counter(lst-ctn-kix_6zwgo5b1uhw7-4,lower-latin) ". "}.lst-kix_u0t4yt1grtwe-4>li:before{content:"" counter(lst-ctn-kix_u0t4yt1grtwe-4,lower-latin) ". "}ol.lst-kix_avu7jq5xuo82-6.start{counter-reset:lst-ctn-kix_avu7jq5xuo82-6 0}.lst-kix_kg9nuuktziog-7>li:before{content:"" counter(lst-ctn-kix_kg9nuuktziog-7,lower-latin) ". "}ol.lst-kix_wljc02q05kpj-2.start{counter-reset:lst-ctn-kix_wljc02q05kpj-2 0}.lst-kix_wd70178zyj44-3>li{counter-increment:lst-ctn-kix_wd70178zyj44-3}.lst-kix_kg9nuuktziog-5>li:before{content:"" counter(lst-ctn-kix_kg9nuuktziog-5,lower-roman) ". "}ol.lst-kix_6zwgo5b1uhw7-6.start{counter-reset:lst-ctn-kix_6zwgo5b1uhw7-6 0}ol.lst-kix_cxfq5ti6eb8z-2.start{counter-reset:lst-ctn-kix_cxfq5ti6eb8z-2 0}.lst-kix_gen1172f8ewp-1>li:before{content:"" counter(lst-ctn-kix_gen1172f8ewp-1,upper-latin) ". "}.lst-kix_gen1172f8ewp-3>li:before{content:"" counter(lst-ctn-kix_gen1172f8ewp-3,lower-latin) ") "}.lst-kix_fplirl6b8ae4-1>li{counter-increment:lst-ctn-kix_fplirl6b8ae4-1}.lst-kix_avu7jq5xuo82-3>li{counter-increment:lst-ctn-kix_avu7jq5xuo82-3}.lst-kix_6zwgo5b1uhw7-0>li:before{content:"" counter(lst-ctn-kix_6zwgo5b1uhw7-0,decimal) ". "}.lst-kix_kg9nuuktziog-1>li:before{content:"" counter(lst-ctn-kix_kg9nuuktziog-1,lower-latin) ". "}.lst-kix_cxfq5ti6eb8z-8>li{counter-increment:lst-ctn-kix_cxfq5ti6eb8z-8}ol.lst-kix_avu7jq5xuo82-5.start{counter-reset:lst-ctn-kix_avu7jq5xuo82-5 0}.lst-kix_kg9nuuktziog-2>li{counter-increment:lst-ctn-kix_kg9nuuktziog-2}.lst-kix_kg9nuuktziog-3>li:before{content:"" counter(lst-ctn-kix_kg9nuuktziog-3,decimal) ". "}.lst-kix_cxfq5ti6eb8z-2>li{counter-increment:lst-ctn-kix_cxfq5ti6eb8z-2}.lst-kix_kg9nuuktziog-8>li{counter-increment:lst-ctn-kix_kg9nuuktziog-8}.lst-kix_6zwgo5b1uhw7-4>li{counter-increment:lst-ctn-kix_6zwgo5b1uhw7-4}ol.lst-kix_kg9nuuktziog-0.start{counter-reset:lst-ctn-kix_kg9nuuktziog-0 0}ol.lst-kix_wd70178zyj44-6.start{counter-reset:lst-ctn-kix_wd70178zyj44-6 0}ol.lst-kix_fplirl6b8ae4-1.start{counter-reset:lst-ctn-kix_fplirl6b8ae4-1 0}.lst-kix_avu7jq5xuo82-4>li{counter-increment:lst-ctn-kix_avu7jq5xuo82-4}.lst-kix_u0t4yt1grtwe-7>li{counter-increment:lst-ctn-kix_u0t4yt1grtwe-7}.lst-kix_6zwgo5b1uhw7-8>li{counter-increment:lst-ctn-kix_6zwgo5b1uhw7-8}ol.lst-kix_avu7jq5xuo82-4.start{counter-reset:lst-ctn-kix_avu7jq5xuo82-4 0}ol.lst-kix_kg9nuuktziog-4{list-style-type:none}ol.lst-kix_bano2ezcy3fm-5.start{counter-reset:lst-ctn-kix_bano2ezcy3fm-5 0}.lst-kix_avu7jq5xuo82-2>li:before{content:"" counter(lst-ctn-kix_avu7jq5xuo82-2,decimal) ". "}ol.lst-kix_kg9nuuktziog-5{list-style-type:none}ol.lst-kix_kg9nuuktziog-2{list-style-type:none}.lst-kix_avu7jq5xuo82-1>li:before{content:"" counter(lst-ctn-kix_avu7jq5xuo82-1,upper-latin) ". "}.lst-kix_avu7jq5xuo82-3>li:before{content:"" counter(lst-ctn-kix_avu7jq5xuo82-3,lower-latin) ") "}ol.lst-kix_kg9nuuktziog-3{list-style-type:none}ol.lst-kix_kg9nuuktziog-0{list-style-type:none}ol.lst-kix_cxfq5ti6eb8z-1.start{counter-reset:lst-ctn-kix_cxfq5ti6eb8z-1 0}.lst-kix_avu7jq5xuo82-0>li:before{content:"" counter(lst-ctn-kix_avu7jq5xuo82-0,upper-roman) ". "}.lst-kix_avu7jq5xuo82-4>li:before{content:"(" counter(lst-ctn-kix_avu7jq5xuo82-4,decimal) ") "}ol.lst-kix_kg9nuuktziog-1{list-style-type:none}ol.lst-kix_wd70178zyj44-5.start{counter-reset:lst-ctn-kix_wd70178zyj44-5 0}ol.lst-kix_gen1172f8ewp-4.start{counter-reset:lst-ctn-kix_gen1172f8ewp-4 0}.lst-kix_fplirl6b8ae4-0>li{counter-increment:lst-ctn-kix_fplirl6b8ae4-0}.lst-kix_avu7jq5xuo82-8>li:before{content:"(" counter(lst-ctn-kix_avu7jq5xuo82-8,lower-roman) ") "}.lst-kix_kg9nuuktziog-7>li{counter-increment:lst-ctn-kix_kg9nuuktziog-7}ol.lst-kix_6zwgo5b1uhw7-8.start{counter-reset:lst-ctn-kix_6zwgo5b1uhw7-8 0}.lst-kix_avu7jq5xuo82-6>li:before{content:"(" counter(lst-ctn-kix_avu7jq5xuo82-6,lower-roman) ") "}.lst-kix_avu7jq5xuo82-5>li:before{content:"(" counter(lst-ctn-kix_avu7jq5xuo82-5,lower-latin) ") "}.lst-kix_avu7jq5xuo82-7>li:before{content:"(" counter(lst-ctn-kix_avu7jq5xuo82-7,lower-latin) ") "}ol.lst-kix_kg9nuuktziog-8{list-style-type:none}.lst-kix_bano2ezcy3fm-2>li{counter-increment:lst-ctn-kix_bano2ezcy3fm-2}ol.lst-kix_wljc02q05kpj-4.start{counter-reset:lst-ctn-kix_wljc02q05kpj-4 0}.lst-kix_u0t4yt1grtwe-5>li{counter-increment:lst-ctn-kix_u0t4yt1grtwe-5}ol.lst-kix_cxfq5ti6eb8z-7.start{counter-reset:lst-ctn-kix_cxfq5ti6eb8z-7 0}ol.lst-kix_kg9nuuktziog-6{list-style-type:none}ol.lst-kix_kg9nuuktziog-7{list-style-type:none}ol.lst-kix_kg9nuuktziog-8.start{counter-reset:lst-ctn-kix_kg9nuuktziog-8 0}ol.lst-kix_fplirl6b8ae4-6.start{counter-reset:lst-ctn-kix_fplirl6b8ae4-6 0}.lst-kix_avu7jq5xuo82-2>li{counter-increment:lst-ctn-kix_avu7jq5xuo82-2}.lst-kix_wd70178zyj44-4>li{counter-increment:lst-ctn-kix_wd70178zyj44-4}.lst-kix_fplirl6b8ae4-2>li{counter-increment:lst-ctn-kix_fplirl6b8ae4-2}ol.lst-kix_bano2ezcy3fm-0{list-style-type:none}ol.lst-kix_bano2ezcy3fm-1{list-style-type:none}ol.lst-kix_fplirl6b8ae4-1{list-style-type:none}ol.lst-kix_fplirl6b8ae4-0{list-style-type:none}ol.lst-kix_bano2ezcy3fm-4{list-style-type:none}ol.lst-kix_bano2ezcy3fm-5{list-style-type:none}ol.lst-kix_bano2ezcy3fm-2{list-style-type:none}ol.lst-kix_bano2ezcy3fm-3{list-style-type:none}ol.lst-kix_bano2ezcy3fm-8{list-style-type:none}.lst-kix_cxfq5ti6eb8z-1>li{counter-increment:lst-ctn-kix_cxfq5ti6eb8z-1}.lst-kix_wd70178zyj44-8>li{counter-increment:lst-ctn-kix_wd70178zyj44-8}ol.lst-kix_bano2ezcy3fm-6{list-style-type:none}ol.lst-kix_bano2ezcy3fm-7{list-style-type:none}ol.lst-kix_kg9nuuktziog-2.start{counter-reset:lst-ctn-kix_kg9nuuktziog-2 0}ol.lst-kix_cxfq5ti6eb8z-6.start{counter-reset:lst-ctn-kix_cxfq5ti6eb8z-6 0}ol.lst-kix_bano2ezcy3fm-6.start{counter-reset:lst-ctn-kix_bano2ezcy3fm-6 0}.lst-kix_6zwgo5b1uhw7-6>li{counter-increment:lst-ctn-kix_6zwgo5b1uhw7-6}.lst-kix_wd70178zyj44-2>li{counter-increment:lst-ctn-kix_wd70178zyj44-2}ol.lst-kix_6zwgo5b1uhw7-2.start{counter-reset:lst-ctn-kix_6zwgo5b1uhw7-2 0}.lst-kix_wljc02q05kpj-8>li{counter-increment:lst-ctn-kix_wljc02q05kpj-8}.lst-kix_kg9nuuktziog-0>li{counter-increment:lst-ctn-kix_kg9nuuktziog-0}.lst-kix_gen1172f8ewp-8>li{counter-increment:lst-ctn-kix_gen1172f8ewp-8}ol.lst-kix_bano2ezcy3fm-0.start{counter-reset:lst-ctn-kix_bano2ezcy3fm-0 0}.lst-kix_u0t4yt1grtwe-0>li{counter-increment:lst-ctn-kix_u0t4yt1grtwe-0}.lst-kix_u0t4yt1grtwe-3>li{counter-increment:lst-ctn-kix_u0t4yt1grtwe-3}.lst-kix_bano2ezcy3fm-0>li{counter-increment:lst-ctn-kix_bano2ezcy3fm-0}ol.lst-kix_wljc02q05kpj-5.start{counter-reset:lst-ctn-kix_wljc02q05kpj-5 0}ol.lst-kix_kg9nuuktziog-3.start{counter-reset:lst-ctn-kix_kg9nuuktziog-3 0}ol.lst-kix_gen1172f8ewp-3.start{counter-reset:lst-ctn-kix_gen1172f8ewp-3 0}ol.lst-kix_fplirl6b8ae4-3{list-style-type:none}ol.lst-kix_fplirl6b8ae4-2{list-style-type:none}ol.lst-kix_fplirl6b8ae4-5{list-style-type:none}ol.lst-kix_6zwgo5b1uhw7-3.start{counter-reset:lst-ctn-kix_6zwgo5b1uhw7-3 0}ol.lst-kix_fplirl6b8ae4-4{list-style-type:none}.lst-kix_bano2ezcy3fm-6>li{counter-increment:lst-ctn-kix_bano2ezcy3fm-6}ol.lst-kix_fplirl6b8ae4-7{list-style-type:none}ol.lst-kix_fplirl6b8ae4-6{list-style-type:none}ol.lst-kix_fplirl6b8ae4-8{list-style-type:none}ol.lst-kix_cxfq5ti6eb8z-1{list-style-type:none}.lst-kix_wljc02q05kpj-4>li{counter-increment:lst-ctn-kix_wljc02q05kpj-4}.lst-kix_wd70178zyj44-0>li:before{content:"" counter(lst-ctn-kix_wd70178zyj44-0,upper-roman) ". "}ol.lst-kix_cxfq5ti6eb8z-0{list-style-type:none}.lst-kix_wljc02q05kpj-2>li:before{content:"" counter(lst-ctn-kix_wljc02q05kpj-2,lower-roman) ". "}ol.lst-kix_cxfq5ti6eb8z-3{list-style-type:none}ol.lst-kix_u0t4yt1grtwe-2.start{counter-reset:lst-ctn-kix_u0t4yt1grtwe-2 0}ol.lst-kix_cxfq5ti6eb8z-2{list-style-type:none}.lst-kix_wd70178zyj44-2>li:before{content:"" counter(lst-ctn-kix_wd70178zyj44-2,decimal) ". "}.lst-kix_wljc02q05kpj-0>li:before{content:"" counter(lst-ctn-kix_wljc02q05kpj-0,decimal) ". "}.lst-kix_wljc02q05kpj-4>li:before{content:"" counter(lst-ctn-kix_wljc02q05kpj-4,lower-latin) ". "}.lst-kix_6zwgo5b1uhw7-2>li{counter-increment:lst-ctn-kix_6zwgo5b1uhw7-2}.lst-kix_pb7y9mdhka33-3>li:before{content:"-  "}.lst-kix_pb7y9mdhka33-7>li:before{content:"-  "}.lst-kix_wd70178zyj44-4>li:before{content:"(" counter(lst-ctn-kix_wd70178zyj44-4,decimal) ") "}ol.lst-kix_cxfq5ti6eb8z-8{list-style-type:none}.lst-kix_wljc02q05kpj-6>li:before{content:"" counter(lst-ctn-kix_wljc02q05kpj-6,decimal) ". "}.lst-kix_bano2ezcy3fm-7>li{counter-increment:lst-ctn-kix_bano2ezcy3fm-7}ol.lst-kix_cxfq5ti6eb8z-5{list-style-type:none}ol.lst-kix_bano2ezcy3fm-1.start{counter-reset:lst-ctn-kix_bano2ezcy3fm-1 0}ol.lst-kix_cxfq5ti6eb8z-4{list-style-type:none}ol.lst-kix_cxfq5ti6eb8z-7{list-style-type:none}.lst-kix_pb7y9mdhka33-5>li:before{content:"-  "}ol.lst-kix_cxfq5ti6eb8z-6{list-style-type:none}ol.lst-kix_fplirl6b8ae4-5.start{counter-reset:lst-ctn-kix_fplirl6b8ae4-5 0}.lst-kix_cxfq5ti6eb8z-4>li{counter-increment:lst-ctn-kix_cxfq5ti6eb8z-4}.lst-kix_gen1172f8ewp-4>li{counter-increment:lst-ctn-kix_gen1172f8ewp-4}.lst-kix_fplirl6b8ae4-8>li:before{content:"" counter(lst-ctn-kix_fplirl6b8ae4-8,lower-roman) ". "}ol.lst-kix_wljc02q05kpj-6.start{counter-reset:lst-ctn-kix_wljc02q05kpj-6 0}.lst-kix_u0t4yt1grtwe-1>li{counter-increment:lst-ctn-kix_u0t4yt1grtwe-1}ol.lst-kix_kg9nuuktziog-4.start{counter-reset:lst-ctn-kix_kg9nuuktziog-4 0}.lst-kix_wljc02q05kpj-5>li{counter-increment:lst-ctn-kix_wljc02q05kpj-5}.lst-kix_pb7y9mdhka33-1>li:before{content:"-  "}ol.lst-kix_u0t4yt1grtwe-2{list-style-type:none}ol.lst-kix_6zwgo5b1uhw7-1.start{counter-reset:lst-ctn-kix_6zwgo5b1uhw7-1 0}ol.lst-kix_wd70178zyj44-1.start{counter-reset:lst-ctn-kix_wd70178zyj44-1 0}ol.lst-kix_u0t4yt1grtwe-1{list-style-type:none}.lst-kix_cxfq5ti6eb8z-6>li{counter-increment:lst-ctn-kix_cxfq5ti6eb8z-6}.lst-kix_fplirl6b8ae4-4>li:before{content:"" counter(lst-ctn-kix_fplirl6b8ae4-4,lower-latin) ". "}ol.lst-kix_u0t4yt1grtwe-0{list-style-type:none}.lst-kix_kg9nuuktziog-4>li{counter-increment:lst-ctn-kix_kg9nuuktziog-4}.lst-kix_fplirl6b8ae4-2>li:before{content:"" counter(lst-ctn-kix_fplirl6b8ae4-2,lower-roman) ". "}.lst-kix_fplirl6b8ae4-6>li:before{content:"" counter(lst-ctn-kix_fplirl6b8ae4-6,decimal) ". "}.lst-kix_bano2ezcy3fm-8>li{counter-increment:lst-ctn-kix_bano2ezcy3fm-8}ol.lst-kix_u0t4yt1grtwe-8{list-style-type:none}ol.lst-kix_u0t4yt1grtwe-7{list-style-type:none}ol.lst-kix_u0t4yt1grtwe-6{list-style-type:none}.lst-kix_u0t4yt1grtwe-2>li{counter-increment:lst-ctn-kix_u0t4yt1grtwe-2}ol.lst-kix_u0t4yt1grtwe-5{list-style-type:none}ol.lst-kix_u0t4yt1grtwe-4{list-style-type:none}ol.lst-kix_u0t4yt1grtwe-3{list-style-type:none}.lst-kix_6zwgo5b1uhw7-0>li{counter-increment:lst-ctn-kix_6zwgo5b1uhw7-0}.lst-kix_wd70178zyj44-8>li:before{content:"(" counter(lst-ctn-kix_wd70178zyj44-8,lower-roman) ") "}ol.lst-kix_avu7jq5xuo82-0.start{counter-reset:lst-ctn-kix_avu7jq5xuo82-0 0}.lst-kix_wd70178zyj44-6>li:before{content:"(" counter(lst-ctn-kix_wd70178zyj44-6,lower-roman) ") "}ol.lst-kix_u0t4yt1grtwe-5.start{counter-reset:lst-ctn-kix_u0t4yt1grtwe-5 0}.lst-kix_wd70178zyj44-1>li{counter-increment:lst-ctn-kix_wd70178zyj44-1}.lst-kix_wljc02q05kpj-3>li{counter-increment:lst-ctn-kix_wljc02q05kpj-3}.lst-kix_fplirl6b8ae4-0>li:before{content:"" counter(lst-ctn-kix_fplirl6b8ae4-0,decimal) ". "}.lst-kix_gen1172f8ewp-3>li{counter-increment:lst-ctn-kix_gen1172f8ewp-3}.lst-kix_fplirl6b8ae4-5>li{counter-increment:lst-ctn-kix_fplirl6b8ae4-5}.lst-kix_u0t4yt1grtwe-5>li:before{content:"" counter(lst-ctn-kix_u0t4yt1grtwe-5,lower-roman) ". "}ol.lst-kix_wljc02q05kpj-8.start{counter-reset:lst-ctn-kix_wljc02q05kpj-8 0}.lst-kix_u0t4yt1grtwe-3>li:before{content:"" counter(lst-ctn-kix_u0t4yt1grtwe-3,decimal) ". "}.lst-kix_u0t4yt1grtwe-7>li:before{content:"" counter(lst-ctn-kix_u0t4yt1grtwe-7,lower-latin) ". "}.lst-kix_6zwgo5b1uhw7-7>li:before{content:"" counter(lst-ctn-kix_6zwgo5b1uhw7-7,lower-latin) ". "}.lst-kix_6zwgo5b1uhw7-3>li:before{content:"" counter(lst-ctn-kix_6zwgo5b1uhw7-3,decimal) ". "}ol.lst-kix_fplirl6b8ae4-7.start{counter-reset:lst-ctn-kix_fplirl6b8ae4-7 0}.lst-kix_u0t4yt1grtwe-1>li:before{content:"" counter(lst-ctn-kix_u0t4yt1grtwe-1,lower-latin) ". "}ol.lst-kix_wd70178zyj44-0.start{counter-reset:lst-ctn-kix_wd70178zyj44-0 0}ol.lst-kix_kg9nuuktziog-6.start{counter-reset:lst-ctn-kix_kg9nuuktziog-6 0}.lst-kix_6zwgo5b1uhw7-1>li:before{content:"" counter(lst-ctn-kix_6zwgo5b1uhw7-1,lower-latin) ". "}.lst-kix_gen1172f8ewp-6>li:before{content:"(" counter(lst-ctn-kix_gen1172f8ewp-6,lower-roman) ") "}.lst-kix_gen1172f8ewp-4>li:before{content:"(" counter(lst-ctn-kix_gen1172f8ewp-4,decimal) ") "}.lst-kix_gen1172f8ewp-8>li:before{content:"(" counter(lst-ctn-kix_gen1172f8ewp-8,lower-roman) ") "}ol.lst-kix_u0t4yt1grtwe-4.start{counter-reset:lst-ctn-kix_u0t4yt1grtwe-4 0}ol.lst-kix_wljc02q05kpj-8{list-style-type:none}ol.lst-kix_bano2ezcy3fm-3.start{counter-reset:lst-ctn-kix_bano2ezcy3fm-3 0}.lst-kix_6zwgo5b1uhw7-5>li:before{content:"" counter(lst-ctn-kix_6zwgo5b1uhw7-5,lower-roman) ". "}ol.lst-kix_wljc02q05kpj-1{list-style-type:none}.lst-kix_wd70178zyj44-6>li{counter-increment:lst-ctn-kix_wd70178zyj44-6}ol.lst-kix_wljc02q05kpj-0{list-style-type:none}.lst-kix_kg9nuuktziog-6>li:before{content:"" counter(lst-ctn-kix_kg9nuuktziog-6,decimal) ". "}.lst-kix_kg9nuuktziog-8>li:before{content:"" counter(lst-ctn-kix_kg9nuuktziog-8,lower-roman) ". "}ol.lst-kix_u0t4yt1grtwe-3.start{counter-reset:lst-ctn-kix_u0t4yt1grtwe-3 0}ol.lst-kix_wljc02q05kpj-3{list-style-type:none}ol.lst-kix_fplirl6b8ae4-8.start{counter-reset:lst-ctn-kix_fplirl6b8ae4-8 0}ol.lst-kix_wljc02q05kpj-2{list-style-type:none}.lst-kix_gen1172f8ewp-0>li:before{content:"" counter(lst-ctn-kix_gen1172f8ewp-0,upper-roman) ". "}ol.lst-kix_wljc02q05kpj-5{list-style-type:none}ol.lst-kix_wljc02q05kpj-4{list-style-type:none}ol.lst-kix_wljc02q05kpj-7{list-style-type:none}ol.lst-kix_wljc02q05kpj-6{list-style-type:none}.lst-kix_gen1172f8ewp-2>li:before{content:"" counter(lst-ctn-kix_gen1172f8ewp-2,decimal) ". "}ol.lst-kix_kg9nuuktziog-7.start{counter-reset:lst-ctn-kix_kg9nuuktziog-7 0}.lst-kix_avu7jq5xuo82-0>li{counter-increment:lst-ctn-kix_avu7jq5xuo82-0}.lst-kix_fplirl6b8ae4-4>li{counter-increment:lst-ctn-kix_fplirl6b8ae4-4}.lst-kix_wd70178zyj44-0>li{counter-increment:lst-ctn-kix_wd70178zyj44-0}ul.lst-kix_pb7y9mdhka33-3{list-style-type:none}ul.lst-kix_pb7y9mdhka33-4{list-style-type:none}.lst-kix_kg9nuuktziog-0>li:before{content:"" counter(lst-ctn-kix_kg9nuuktziog-0,decimal) ". "}.lst-kix_6zwgo5b1uhw7-7>li{counter-increment:lst-ctn-kix_6zwgo5b1uhw7-7}ul.lst-kix_pb7y9mdhka33-5{list-style-type:none}.lst-kix_avu7jq5xuo82-6>li{counter-increment:lst-ctn-kix_avu7jq5xuo82-6}ul.lst-kix_pb7y9mdhka33-6{list-style-type:none}ul.lst-kix_pb7y9mdhka33-0{list-style-type:none}.lst-kix_wljc02q05kpj-8>li:before{content:"" counter(lst-ctn-kix_wljc02q05kpj-8,lower-roman) ". "}ul.lst-kix_pb7y9mdhka33-1{list-style-type:none}ol.lst-kix_bano2ezcy3fm-4.start{counter-reset:lst-ctn-kix_bano2ezcy3fm-4 0}ul.lst-kix_pb7y9mdhka33-2{list-style-type:none}ol.lst-kix_gen1172f8ewp-0.start{counter-reset:lst-ctn-kix_gen1172f8ewp-0 0}ol.lst-kix_cxfq5ti6eb8z-8.start{counter-reset:lst-ctn-kix_cxfq5ti6eb8z-8 0}.lst-kix_kg9nuuktziog-2>li:before{content:"" counter(lst-ctn-kix_kg9nuuktziog-2,lower-roman) ". "}.lst-kix_kg9nuuktziog-4>li:before{content:"" counter(lst-ctn-kix_kg9nuuktziog-4,lower-latin) ". "}.lst-kix_6zwgo5b1uhw7-1>li{counter-increment:lst-ctn-kix_6zwgo5b1uhw7-1}ul.lst-kix_pb7y9mdhka33-7{list-style-type:none}ul.lst-kix_pb7y9mdhka33-8{list-style-type:none}.lst-kix_cxfq5ti6eb8z-5>li{counter-increment:lst-ctn-kix_cxfq5ti6eb8z-5}.lst-kix_kg9nuuktziog-5>li{counter-increment:lst-ctn-kix_kg9nuuktziog-5}ol{margin:0;padding:0}table td,table th{padding:0}.c22{border-right-style:solid;padding:5pt 5pt 5pt 5pt;border-bottom-color:#ffffff;border-top-width:1pt;border-right-width:1pt;border-left-color:#ffffff;vertical-align:top;border-right-color:#ffffff;border-left-width:1pt;border-top-style:solid;border-left-style:solid;border-bottom-width:1pt;width:225.7pt;border-top-color:#ffffff;border-bottom-style:solid}.c3{background-color:#efefef;color:#000000;font-weight:400;text-decoration:none;vertical-align:baseline;font-size:10pt;font-family:"Consolas";font-style:normal}.c0{background-color:#ffffff;color:#000000;font-weight:700;text-decoration:none;vertical-align:baseline;font-size:10pt;font-family:"Lora";font-style:normal}.c28{color:#222222;font-weight:400;text-decoration:none;vertical-align:baseline;font-size:10pt;font-family:"Arial";font-style:normal}.c36{color:#000000;font-weight:400;text-decoration:none;vertical-align:baseline;font-size:11pt;font-family:"Arial";font-style:normal}.c20{color:#000000;font-weight:700;text-decoration:none;vertical-align:baseline;font-size:10pt;font-family:"Lora";font-style:normal}.c15{padding-top:0pt;padding-bottom:0pt;line-height:1.15;orphans:2;widows:2;text-align:center;height:11pt}.c9{color:#000000;font-weight:400;text-decoration:none;vertical-align:baseline;font-size:8pt;font-family:"Lora";font-style:normal}.c2{padding-top:0pt;padding-bottom:0pt;line-height:1.15;orphans:2;widows:2;text-align:justify;height:11pt}.c1{color:#000000;font-weight:400;text-decoration:none;vertical-align:baseline;font-size:10pt;font-family:"Lora";font-style:normal}.c6{-webkit-text-decoration-skip:none;color:#1155cc;font-weight:400;text-decoration:underline;text-decoration-skip-ink:none;font-size:8pt;font-family:"Lora"}.c4{padding-top:0pt;padding-bottom:0pt;line-height:1.15;orphans:2;widows:2;text-align:justify}.c5{padding-top:0pt;padding-bottom:0pt;line-height:1.15;orphans:2;widows:2;text-align:left}.c26{padding-top:0pt;padding-bottom:0pt;line-height:1.15;orphans:2;widows:2;text-align:center}.c39{padding-top:12pt;padding-bottom:12pt;line-height:1.15;orphans:2;widows:2;text-align:left}.c7{padding-top:0pt;padding-bottom:0pt;line-height:1.0;text-align:justify;height:11pt}.c32{padding-top:0pt;padding-bottom:0pt;line-height:1.0;text-align:left}.c13{background-color:#efefef;font-size:10pt;font-family:"Consolas";font-weight:400}.c25{-webkit-text-decoration-skip:none;color:#1155cc;text-decoration:underline;text-decoration-skip-ink:none}.c10{background-color:#ffffff;font-size:10pt;font-family:"Lora";font-weight:400}.c30{border-spacing:0;border-collapse:collapse;margin-right:auto}.c16{-webkit-text-decoration-skip:none;color:#642a8f;text-decoration:underline;text-decoration-skip-ink:none}.c12{padding-top:0pt;padding-bottom:0pt;line-height:1.0;text-align:justify}.c34{font-size:10pt;font-family:"Consolas";font-weight:400}.c31{font-weight:400;font-size:12pt;font-family:"Times New Roman"}.c18{font-size:8pt;font-family:"Lora";font-weight:400}.c24{font-size:10pt;font-family:"Lora";font-weight:400}.c23{font-size:10pt;font-family:"Lora";font-weight:700}.c37{color:#000000;text-decoration:none;vertical-align:baseline}.c19{padding:0;margin:0}.c11{margin-left:36pt;padding-left:0pt}.c8{color:inherit;text-decoration:inherit}.c29{max-width:451.4pt;padding:42.5pt 72pt 72pt 72pt}.c21{height:11pt}.c27{margin-left:36pt}.c17{font-style:italic}.c33{vertical-align:super}.c14{background-color:#ffffff}.c35{text-indent:36pt}.c38{height:212pt}.title{padding-top:0pt;color:#000000;font-size:26pt;padding-bottom:3pt;font-family:"Arial";line-height:1.15;page-break-after:avoid;orphans:2;widows:2;text-align:left}.subtitle{padding-top:0pt;color:#666666;font-size:15pt;padding-bottom:16pt;font-family:"Arial";line-height:1.15;page-break-after:avoid;orphans:2;widows:2;text-align:left}li{color:#000000;font-size:11pt;font-family:"Arial"}p{margin:0;color:#000000;font-size:11pt;font-family:"Arial"}h1{padding-top:20pt;color:#000000;font-size:20pt;padding-bottom:6pt;font-family:"Arial";line-height:1.15;page-break-after:avoid;orphans:2;widows:2;text-align:left}h2{padding-top:18pt;color:#000000;font-size:16pt;padding-bottom:6pt;font-family:"Arial";line-height:1.15;page-break-after:avoid;orphans:2;widows:2;text-align:left}h3{padding-top:16pt;color:#434343;font-size:14pt;padding-bottom:4pt;font-family:"Arial";line-height:1.15;page-break-after:avoid;orphans:2;widows:2;text-align:left}h4{padding-top:14pt;color:#666666;font-size:12pt;padding-bottom:4pt;font-family:"Arial";line-height:1.15;page-break-after:avoid;orphans:2;widows:2;text-align:left}h5{padding-top:12pt;color:#666666;font-size:11pt;padding-bottom:4pt;font-family:"Arial";line-height:1.15;page-break-after:avoid;orphans:2;widows:2;text-align:left}h6{padding-top:12pt;color:#666666;font-size:11pt;padding-bottom:4pt;font-family:"Arial";line-height:1.15;page-break-after:avoid;font-style:italic;orphans:2;widows:2;text-align:left}</style></head><body class="c14 c29"><p class="c4"><span class="c1">GWAS - genome-wide association search - a study aimed at finding relationships between the phenotype (for example, disease) of an organism and a set of SNPs, deletions, insertions and other individual characteristics of the genome (down to individual genes). The analysis is usually performed on two samples. The test sample has the phenotype of interest and the control sample is collected from people of the same sex, age, etc. Today this approach is used to search for associated morbidity of COVID-19. </span></p><p class="c4"><span class="c1">Our work pursued two goals:</span></p><ol class="c19 lst-kix_cxfq5ti6eb8z-0 start" start="1"><li class="c4 c11"><span class="c1">Familiarization the PLINK tool for GWAS analysis</span></li><li class="c4 c11"><span class="c1">Comparative analysis of articles on GWAS studies of patients with COVID-19</span></li></ol><p class="c5 c21"><span class="c1"></span></p><p class="c5"><span class="c20">Step 1. How to work with PLINK</span></p><p class="c5 c21"><span class="c20"></span></p><p class="c5"><span class="c20">Software</span></p><ul class="c19 lst-kix_pb7y9mdhka33-0 start"><li class="c4 c11"><span class="c10">PLINK version 1.09 [</span><span class="c25 c10"><a class="c8" href="https://www.google.com/url?q=https://doi.org/10.1017/S0033291714002451&amp;sa=D&amp;ust=1608477671562000&amp;usg=AOvVaw3Xvu6TqPoZmJMPxskvJ0HT">1</a></span><span class="c10">]</span><span class="c10">, </span><span class="c10">&nbsp;downloaded from </span><span class="c10 c16"><a class="c8" href="https://www.google.com/url?q=http://zzz.bwh.harvard.edu/plink/&amp;sa=D&amp;ust=1608477671563000&amp;usg=AOvVaw1bBTyGJRxPlWwMOv4FVt8Z">http://zzz.bwh.harvard.edu/plink/</a></span><span class="c23">: </span><span class="c10">QC procedures and statistical analyses</span></li><li class="c4 c11"><span class="c24">R version 3.6.3</span><span class="c23">, </span><span class="c10 c16"><a class="c8" href="https://www.google.com/url?q=https://www.r-project.org/&amp;sa=D&amp;ust=1608477671564000&amp;usg=AOvVaw14jKfUheDijF12zf0Nq9b3">https://w</a></span><span class="c10 c16"><a class="c8" href="https://www.google.com/url?q=https://www.r-project.org/&amp;sa=D&amp;ust=1608477671564000&amp;usg=AOvVaw14jKfUheDijF12zf0Nq9b3">ww.r-project.org/</a></span><span class="c23">&nbsp;(</span><span class="c10">for correct operation it is recommended to use versions&gt; 3.0.0): </span><span class="c10">generated graphs by the GitHub example scripts</span><span class="c10">&nbsp;(</span><span class="c16 c14 c31"><a class="c8" href="https://www.google.com/url?q=https://github.com/MareesAT/GWA_tutorial/&amp;sa=D&amp;ust=1608477671564000&amp;usg=AOvVaw3AK6A2RbF4ERCgBlJFeVqZ">https://github.com/MareesAT/GWA_tutorial/</a></span><span class="c20">) </span></li></ul><p class="c2"><span class="c20"></span></p><p class="c4"><span class="c0">Data </span></p><p class="c4"><span class="c10">To be able to illustrate all analysis steps using realistic genetic data, we simulated a dataset (N = 207) with a binary outcome measure using the publicly available data from the International HapMap Project [</span><span class="c25 c10"><a class="c8" href="https://www.google.com/url?q=http://dx.doi.org/10.1038/nature02168&amp;sa=D&amp;ust=1608477671565000&amp;usg=AOvVaw2LNWjFhPwGfhoUiNg-q01k">2</a></span><span class="c10">]. Here in order to create an ethnically homogenous dataset, Utah residents with ancestry from Northern and Western Europe (CEU) were only included. Because of the relatively small sample size of the HapMap data, genetic effect sizes in these simulations were set at values larger than usually observed in genetic studies of complex traits. It is important to note that larger sample sizes (e.g., at least in the order of thousands but likely even tens or hundreds of thousands) will be required to detect genetic risk factors of complex traits. </span></p><p class="c2 c27"><span class="c1 c14"></span></p><p class="c5"><span class="c20">Quality Control of genetic data</span></p><p class="c4"><span class="c1">The seven QC steps consist of filtering out of SNPs and individuals based on the following: (1) individual and SNP missingness, (2) inconsistencies in assigned and genetic sex of subjects (see sex discrepancy), (3) minor allele frequency (MAF), (4) deviations from Hardy&ndash;Weinberg equilibrium (HWE), (5) heterozygosity rate, (6) relatedness, and (7) ethnic outliers. </span></p><p class="c2"><span class="c1"></span></p><ol class="c19 lst-kix_wljc02q05kpj-0 start" start="1"><li class="c4 c11"><span class="c20">Individual and SNP missingness</span></li></ol><p class="c4"><span class="c1">Individual&#8208;level missingness -- this is the number of SNPs that are missing for a specific individual. High levels of missingness can be an indication of poor DNA quality or technical problems.</span></p><p class="c4"><span class="c1">SNP&#8208;level missingness -- this is the number of individuals in the sample for whom information on a specific SNP is missing. SNPs with a high level of missingness can potentially lead to bias.</span></p><p class="c4"><span class="c1">Investigate missingness per individual and per SNP and make histograms:</span></p><p class="c4"><span class="c13">plink --bfile HapMap_3_r3_1 --missing</span><span class="c24">&nbsp; &nbsp;</span><span class="c1">&nbsp;</span></p><p class="c4"><span class="c1">output: plink.imiss and plink.lmiss, these files show respectively the proportion of missing SNPs per individual and the proportion of missing individuals per SNP &nbsp;</span></p><p class="c4"><span class="c1">Generate plots to visualize the missingness results (Fig.1):</span></p><p class="c2"><span class="c1"></span></p><p class="c4"><span class="c3">Rscript hist_miss.R</span></p><p class="c5"><span style="overflow: hidden; display: inline-block; margin: 0.00px 0.00px; border: 0.00px solid #000000; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px); width: 540.50px; height: 258.75px;"><img alt="" src="images/image17.png" style="width: 540.50px; height: 258.75px; margin-left: 0.00px; margin-top: 0.00px; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px);" title=""></span></p><p class="c5 c21"><span class="c9"></span></p><p class="c26"><span class="c9">Fig. 1. Histograms of individual and SNP missingness</span></p><p class="c15"><span class="c9"></span></p><p class="c4"><span class="c1">Next, we need to delete SNPs and individuals with high levels of missingness. For these two commands are used:</span></p><p class="c4"><span class="c13">-</span><span class="c13">-geno</span><span class="c1">: excludes SNPs that are missing in a large proportion of the subjects. In this step, SNPs with low genotype calls are removed.&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;It is recommended to first filter SNPs and individuals based on a relaxed threshold (0.2; &gt;20%), as this will filter out SNPs and individuals with very high levels of missingness. Then a filter with a more stringent threshold can be applied (0.02)</span></p><p class="c4"><span class="c13">--mind</span><span class="c1">: excludes individuals who have high rates of genotype missingness. In this step, individuals with low genotype calls are removed. SNP filtering should be performed before individual filtering. </span></p><p class="c2"><span class="c1"></span></p><p class="c4"><span class="c1">Delete SNPs with missingness &gt;0.2:</span></p><p class="c4"><span class="c3">plink --bfile HapMap_3_r3_1 --geno 0.2 --make-bed --out HapMap_3_r3_2</span></p><p class="c2"><span class="c1"></span></p><p class="c4"><span class="c1">Delete individuals with missingness &gt;0.2:</span></p><p class="c4"><span class="c3">plink --bfile HapMap_3_r3_2 --mind 0.2 --make-bed --out HapMap_3_r3_3</span></p><p class="c2"><span class="c1"></span></p><p class="c4"><span class="c1">Delete SNPs with missingness &gt;0.02:</span></p><p class="c4"><span class="c3">plink --bfile HapMap_3_r3_3 --geno 0.02 --make-bed --out HapMap_3_r3_4</span></p><p class="c2"><span class="c1"></span></p><p class="c4"><span class="c1">Delete individuals with missingness &gt;0.02:</span></p><p class="c4"><span class="c3">plink --bfile HapMap_3_r3_4 --mind 0.02 --make-bed --out HapMap_3_r3_5</span></p><p class="c2"><span class="c3"></span></p><ol class="c19 lst-kix_wljc02q05kpj-0" start="2"><li class="c4 c11"><span class="c20">Check for sex discrepancy</span></li></ol><p class="c4"><span class="c1">Sex discrepancy -- this is the difference between the assigned sex and the sex determined based on the genotype. A discrepancy likely points to sample mix&#8208;ups in the lab. Note, this test can only be conducted when SNPs on the sex chromosomes (X and Y) have been assessed. </span></p><p class="c4"><span class="c24">We will use the command </span><span class="c13">--</span><span class="c13">check-sex</span><span class="c1">, which checks for discrepancies between sex of the individuals recorded in the dataset and their sex based on X chromosome heterozygosity/homozygosity rates. Subjects who were a priori determined as females must have a F-value of &lt;0.2, and subjects who were a priori determined as males must have a F-value &gt;0.8. This F-value is based on the X chromosome inbreeding (homozygosity) estimate. Subjects who do not fulfil these requirements are flagged &quot;PROBLEM&quot; by PLINK.</span></p><p class="c2"><span class="c1"></span></p><p class="c4"><span class="c13">plink --bfile HapMap_3_r3_5 --check-sex</span><span class="c1">&nbsp;</span></p><p class="c2"><span class="c1"></span></p><p class="c4"><span class="c24">Generate plots to visualize the</span><span class="c24">&nbsp;sex-check results (Fig. 2):</span></p><p class="c4"><span class="c3">Rscript gender_check.R</span></p><p class="c4"><span style="overflow: hidden; display: inline-block; margin: 0.00px 0.00px; border: 0.00px solid #000000; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px); width: 601.70px; height: 197.33px;"><img alt="" src="images/image15.png" style="width: 601.70px; height: 197.33px; margin-left: 0.00px; margin-top: 0.00px; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px);" title=""></span></p><p class="c4"><span class="c9">Fig. 2. Sex-check results for all data and for subsets of men an women. Subjects who were a priori determined as females must have a F-value of &lt;0.2, and subjects who were a priori determined as males must have a F-value &gt;0.8. This F-value is based on the X chromosome inbreeding (homozygosity) estimate.</span></p><p class="c2"><span class="c1"></span></p><p class="c4"><span class="c1">These checks indicate that there is one woman with a sex discrepancy, F value of 0.99. (When using other datasets often a few discrepancies will be found). </span></p><p class="c4"><span class="c1">Next, we will delete individuals with sex discrepancy.</span></p><p class="c4"><span class="c3">grep &quot;PROBLEM&quot; plink.sexcheck| awk &#39;{print$1,$2}&#39;&gt; sex_discrepancy.txt</span></p><p class="c4"><span class="c1">This command generates a list of individuals with the status PROBLEM.</span></p><p class="c4"><span class="c13">plink --bfile HapMap_3_r3_5 --remove sex_discrepancy.txt --make-bed --out HapMap_3_r3_6</span><span class="c1">&nbsp;</span></p><p class="c4"><span class="c1">This command removes the list of individuals with the status PROBLEM.</span></p><p class="c2"><span class="c1"></span></p><ol class="c19 lst-kix_wljc02q05kpj-0" start="3"><li class="c4 c11"><span class="c20">Generate a bfile with autosomal SNPs only and delete SNPs with a low minor allele frequency (MAF)</span></li></ol><p class="c4"><span class="c1">Minor allele frequency (MAF) -- this is the frequency of the least often occurring allele at a specific location. Most studies are underpowered to detect associations with SNPs with a low MAF and therefore exclude these SNPs.</span></p><p class="c4"><span class="c24">SNPs with a low MAF are rare, therefore power is lacking for detecting SNP&#8208;phenotype associations. These SNPs are also more prone to genotyping errors. The MAF threshold should depend on the sample size, larger samples can use lower MAF thresholds. Respectively, for large (N = 100.000) vs. moderate samples (N = 10000), 0.01 and 0.05 are commonly used as MAF threshold. </span></p><p class="c4"><span class="c24">The command </span><span class="c13">--maf</span><span class="c1">&nbsp;is used for including in the following analysis only SNPs above the set MAF threshold. </span></p><p class="c4"><span class="c1">Select autosomal SNPs only (i.e., from chromosomes 1 to 22):</span></p><p class="c4"><span class="c3">awk &#39;{ if ($1 &gt;= 1 &amp;&amp; $1 &lt;= 22) print $2 }&#39; HapMap_3_r3_6.bim &gt; snp_1_22.txt</span></p><p class="c4"><span class="c3">plink --bfile HapMap_3_r3_6 --extract snp_1_22.txt --make-bed --out HapMap_3_r3_7</span></p><p class="c2"><span class="c1"></span></p><p class="c4"><span class="c1">Generate a plot of the MAF distribution (Fig. 3):</span></p><p class="c4"><span class="c3">plink --bfile HapMap_3_r3_7 --freq --out MAF_check</span></p><p class="c4"><span class="c3">Rscript MAF_check.R</span></p><p class="c4"><span style="overflow: hidden; display: inline-block; margin: 0.00px 0.00px; border: 0.00px solid #000000; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px); width: 271.00px; height: 257.69px;"><img alt="" src="images/image5.png" style="width: 271.00px; height: 274.00px; margin-left: 0.00px; margin-top: -16.31px; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px);" title=""></span></p><p class="c4"><span class="c9">Fig. 3. MAF distribution. SNPs with a low MAF are rare, therefore power is lacking for detecting SNP&#8208;phenotype associations. These SNPs are also more prone to genotyping errors.</span></p><p class="c2"><span class="c1"></span></p><p class="c4"><span class="c1">Remove SNPs with a low MAF frequency:</span></p><p class="c4"><span class="c3">plink --bfile HapMap_3_r3_7 --maf 0.05 --make-bed --out HapMap_3_r3_8 </span></p><p class="c4"><span class="c1">1073226 SNPs are left</span></p><p class="c2"><span class="c1"></span></p><ol class="c19 lst-kix_wljc02q05kpj-0" start="4"><li class="c4 c11"><span class="c20">Hardy&ndash;Weinberg equilibrium (HWE)</span></li></ol><p class="c4"><span class="c1">The Hardy&ndash;Weinberg (dis)equilibrium (HWE) law concerns the relation between the allele and genotype frequencies. It assumes an indefinitely large population, with no selection, mutation, or migration. The law states that the genotype and the allele frequencies are constant over generations. Violation of the HWE law indicates that genotype frequencies are significantly different from expectations (e.g., if the frequency of allele A = 0.20 and the frequency of allele T = 0.80; the expected frequency of genotype AT is 2*0.2*0.8 = 0.32) and the observed frequency should not be significantly different. In GWAS, it is generally assumed that deviations from HWE are the result of genotyping errors. The HWE thresholds in cases are often less stringent than those in controls, as the violation of the HWE law in cases can be indicative of true genetic association with disease risk.</span></p><p class="c4"><span class="c24">Command </span><span class="c13">--hwe</span><span class="c1">&nbsp;excludes markers which deviate from Hardy&ndash;Weinberg equilibrium. For binary traits, it is proposed to exclude: the value of HWE p-value &lt;1e-10 in cases and &lt;1e-6 in the control. A less strict case threshold avoids exclusion of disease-related SNPs in selection. For quantitative traits, a p HWE value &lt;1e-6 is recommended.</span></p><p class="c2"><span class="c1"></span></p><p class="c4"><span class="c1">Check the distribution of HWE p-values of all SNPs:</span></p><p class="c4"><span class="c3">plink --bfile HapMap_3_r3_8 --hardy</span></p><p class="c2"><span class="c1"></span></p><p class="c4"><span class="c1">Selecting SNPs with HWE p-value below 0.00001, required for one of the two plots generated by the next Rscript, allows zooming in on strongly deviating SNPs (Fig. 4). </span></p><p class="c2"><span class="c1"></span></p><p class="c4"><span class="c3">awk &#39;{ if ($9 &lt;0.00001) print $0 }&#39; plink.hwe &gt; plinkzoomhwe.hwe</span></p><p class="c4"><span class="c3">Rscript hwe.R</span></p><p class="c4"><span style="overflow: hidden; display: inline-block; margin: 0.00px 0.00px; border: 0.00px solid #000000; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px); width: 601.70px; height: 220.00px;"><img alt="" src="images/image12.png" style="width: 601.70px; height: 220.00px; margin-left: 0.00px; margin-top: 0.00px; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px);" title=""></span></p><p class="c26"><span class="c18">Fig. 4. The distribution of HWE p-values of all SNPs and strongly deviating SNPs.</span><span class="c1">&nbsp;</span></p><p class="c2"><span class="c1"></span></p><p class="c4"><span class="c24">By default the </span><span class="c13">--hwe</span><span class="c1">&nbsp;option in plink only filters for controls.</span></p><p class="c4"><span class="c1">Therefore, we use two steps, first we use a stringent HWE threshold for controls, followed by a less stringent threshold for the case data:</span></p><p class="c4"><span class="c3">plink --bfile HapMap_3_r3_8 --hwe 1e-6 --make-bed --out HapMap_hwe_filter_step1</span></p><p class="c2"><span class="c1"></span></p><p class="c4"><span class="c1">The HWE threshold for the cases filters out only SNPs which deviate extremely from HWE. This second HWE step only focuses on cases because in the controls all SNPs with a HWE p-value &lt; hwe 1e-6 were already removed:</span></p><p class="c4"><span class="c3">plink --bfile HapMap_hwe_filter_step1 --hwe 1e-10 --hwe-all --make-bed --out HapMap_3_r3_9</span></p><p class="c2"><span class="c1"></span></p><p class="c2"><span class="c1"></span></p><ol class="c19 lst-kix_wljc02q05kpj-0" start="5"><li class="c4 c11"><span class="c20">Heterozygosity </span></li></ol><p class="c4"><span class="c1">Heterozygosity -- this is the carrying of two different alleles of a specific SNP. The heterozygosity rate of an individual is the proportion of heterozygous genotypes. High levels of heterozygosity within an individual might be an indication of low sample quality whereas low levels of heterozygosity may be due to inbreeding.</span></p><p class="c4"><span class="c1">We will exclude individuals with high or low heterozygosity rates. It is recommended to remove individuals who deviate &plusmn;3 SD from the samples&#39; heterozygosity rate mean.</span></p><p class="c4"><span class="c1">Generate a plot of the distribution of the heterozygosity rate of your subjects.</span></p><p class="c4"><span class="c1">Checks for heterozygosity are performed on a set of SNPs which are not highly correlated.</span></p><p class="c4"><span class="c24">Therefore, to generate a list of non-(highly)correlated SNPs, we exclude high inversion regions (inversion.txt [High LD regions]) and prune the SNPs using the command </span><span class="c13">--indep-pairwise</span><span class="c1">.</span></p><p class="c4"><span class="c1">The parameters 50 5 0.2 stand respectively for: the window size, the number of SNPs to shift the window at each step, and the multiple correlation coefficient for a SNP being regressed on all other SNPs simultaneously:</span></p><p class="c2"><span class="c1"></span></p><p class="c4"><span class="c3">plink --bfile HapMap_3_r3_9 --exclude inversion.txt --range --indep-pairwise 50 5 0.2 --out indepSNP</span></p><p class="c4"><span class="c3">plink --bfile HapMap_3_r3_9 --extract indepSNP.prune.in --het --out R_check</span></p><p class="c2"><span class="c1"></span></p><p class="c4"><span class="c1">Plot of the heterozygosity rate distribution (Fig. 5):</span></p><p class="c4"><span class="c3">Rscript check_heterozygosity_rate.R</span></p><p class="c5"><span style="overflow: hidden; display: inline-block; margin: 0.00px 0.00px; border: 0.00px solid #000000; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px); width: 256.00px; height: 231.69px;"><img alt="" src="images/image3.png" style="width: 256.00px; height: 246.00px; margin-left: 0.00px; margin-top: -14.31px; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px);" title=""></span></p><p class="c5"><span class="c9">Fig. 5. Heterozygosity rate</span></p><p class="c5 c21"><span class="c9"></span></p><p class="c4"><span class="c1">The following code generates a list of individuals who deviate more than 3 standard deviations from the heterozygosity rate mean:</span></p><p class="c2"><span class="c1"></span></p><p class="c4"><span class="c3">Rscript heterozygosity_outliers_list.R</span></p><p class="c2"><span class="c1"></span></p><p class="c4"><span class="c1">The output of the command above: fail-het-qc.txt.</span></p><p class="c4"><span class="c1">When using our example data/the HapMap data this list contains 2 individuals (i.e., two individuals have a heterozygosity rate deviating more than 3 SD&#39;s from the mean).</span></p><p class="c4"><span class="c1">Adapt this file to make it compatible for PLINK, by removing all quotation marks from the file and selecting only the first two columns:</span></p><p class="c4"><span class="c3">sed &#39;s/&quot;// g&#39; fail-het-qc.txt | awk &#39;{print$1, $2}&#39;&gt; het_fail_ind.txt</span></p><p class="c2"><span class="c1"></span></p><p class="c4"><span class="c1">Remove heterozygosity rate outliers:</span></p><p class="c4"><span class="c3">plink --bfile HapMap_3_r3_9 --remove het_fail_ind.txt --make-bed --out HapMap_3_r3_10</span></p><p class="c2"><span class="c1"></span></p><ol class="c19 lst-kix_wljc02q05kpj-0" start="6"><li class="c4 c11"><span class="c20">Relatedness</span></li></ol><p class="c4"><span class="c1">Relatedness indicates how strongly a pair of individuals is genetically related. A conventional GWAS assumes that all subjects are unrelated (i.e., no pair of individuals is more closely related than second&#8208;degree relatives). Without appropriate correction, the inclusion of relatives could lead to biased estimations of standard errors of SNP effect sizes. Note that specific tools for analysing family data have been developed.</span></p><p class="c4"><span class="c1">Here we will use two commands:</span></p><p class="c4"><span class="c13">--genome</span><span class="c1">: calculates identity by descent (IBD) of all sample pairs. We need to use &nbsp;independent SNPs (pruning) for this analysis and limit it to autosomal chromosomes only.</span></p><p class="c4"><span class="c13">--min</span><span class="c24">: sets the threshold and creates a list of individuals with relatedness above the chosen threshold. Meaning that subjects who are related at, for example, pi&#8208;hat &gt;0.2 (i.e., second degree relatives) can be detected. Cryptic relatedness can interfere with the association analysis. If we have a family&#8208;based sample (e.g., parent&#8208;offspring), we do not need to remove related pairs but the statistical analysis should take family relatedness into account. However, for a population-based sample, it is suggested to use a pi&#8208;hat threshold of 0.2, which is in line with the literature [</span><span class="c25 c24"><a class="c8" href="https://www.google.com/url?q=https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3025522/&amp;sa=D&amp;ust=1608477671578000&amp;usg=AOvVaw0A5-uT02iLHTZIJRsfG5Pq">4</a></span><span class="c1">].</span></p><p class="c2"><span class="c1"></span></p><p class="c4"><span class="c1">Check for relationships between individuals with a pihat &gt; 0.2:</span></p><p class="c4"><span class="c3">plink --bfile HapMap_3_r3_10 --extract indepSNP.prune.in --genome --min 0.2 --out pihat_min0.2</span></p><p class="c2"><span class="c1"></span></p><p class="c4"><span class="c1">The HapMap dataset is known to contain parent-offspring relations. </span></p><p class="c4"><span class="c1">The following commands will visualize specifically these parent-offspring relations, using the z values:</span></p><p class="c4"><span class="c3">awk &#39;{ if ($8 &gt;0.9) print $0 }&#39; pihat_min0.2.genome&gt;zoom_pihat.genome</span></p><p class="c2"><span class="c1"></span></p><p class="c4"><span class="c1">Generate a plot to assess the type of relationship (Fig. 6):</span></p><p class="c4"><span class="c3">Rscript Relatedness.R</span></p><p class="c2"><span class="c9"></span></p><p class="c4"><span style="overflow: hidden; display: inline-block; margin: 0.00px 0.00px; border: 0.00px solid #000000; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px); width: 293.50px; height: 237.00px;"><img alt="" src="images/image8.png" style="width: 293.50px; height: 237.00px; margin-left: 0.00px; margin-top: 0.00px; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px);" title=""></span></p><p class="c2"><span class="c1"></span></p><p class="c2"><span class="c1"></span></p><p class="c2"><span class="c1"></span></p><p class="c2"><span class="c1"></span></p><p class="c2"><span class="c1"></span></p><p class="c4"><span style="overflow: hidden; display: inline-block; margin: 0.00px 0.00px; border: 0.00px solid #000000; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px); width: 291.00px; height: 265.51px;"><img alt="" src="images/image9.png" style="width: 291.00px; height: 265.51px; margin-left: 0.00px; margin-top: 0.00px; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px);" title=""></span><span style="overflow: hidden; display: inline-block; margin: 0.00px 0.00px; border: 0.00px solid #000000; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px); width: 290.50px; height: 262.63px;"><img alt="" src="images/image10.png" style="width: 290.50px; height: 262.63px; margin-left: 0.00px; margin-top: 0.00px; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px);" title=""></span></p><p class="c26"><span class="c9">Fig. 6. Histogram of relatedness and parent-offspring relations (used the z values).</span></p><p class="c15"><span class="c9"></span></p><p class="c4"><span class="c1">The generated plots show a considerable amount of related individuals (explentation plot; PO = parent-offspring, UN = unrelated individuals) in the Hapmap data, this is expected since the dataset was constructed as such. Normally, family based data should be analyzed using specific family based methods. To demonstrate that the majority of the relatedness was due to parent-offspring we only include founders (individuals without parents in the dataset):</span></p><p class="c2"><span class="c1"></span></p><p class="c4"><span class="c3">plink --bfile HapMap_3_r3_10 --filter-founders --make-bed --out HapMap_3_r3_11</span></p><p class="c2"><span class="c1"></span></p><p class="c4"><span class="c1">Now we will look again for individuals with a pihat &gt;0.2:</span></p><p class="c4"><span class="c3">plink --bfile HapMap_3_r3_11 --extract indepSNP.prune.in --genome --min 0.2 --out pihat_min0.2_in_founders</span></p><p class="c4"><span class="c1">The file &#39;pihat_min0.2_in_founders.genome&#39; shows that, after exclusion of all non-founders, only 1 individual pair with a pihat greater than 0.2 remains in the HapMap data.</span></p><p class="c4"><span class="c1">This is likely to be a full sib or DZ twin pair based on the Z values. Noteworthy, they were not given the same family identity (FID) in the HapMap data.</span></p><p class="c2"><span class="c1"></span></p><p class="c4"><span class="c1">For each pair of &#39;related&#39; individuals with a pihat &gt; 0.2, it is recommended to remove the individual with the lowest call rate:</span></p><p class="c4"><span class="c3">plink --bfile HapMap_3_r3_11 --missing</span></p><p class="c2"><span class="c3"></span></p><p class="c4"><span class="c1">Use an UNIX text editor vim to check which individual has the highest call rate in the &#39;related pair&#39;. </span></p><p class="c2"><span class="c1"></span></p><p class="c4"><span class="c1">Generate a list of FID and IID of the individual(s) with a Pihat above 0.2, to check who had the lower call rate of the pair.</span></p><p class="c4"><span class="c1">In our dataset the individual 13291 &nbsp;NA07045 had the lower call rate.</span></p><p class="c4"><span class="c3">vi 0.2_low_call_rate_pihat.txt</span></p><p class="c4"><span class="c3">i </span></p><p class="c4"><span class="c3">13291 &nbsp;NA07045</span></p><p class="c4"><span class="c3">ESC</span></p><p class="c4"><span class="c3">:wq</span></p><p class="c2"><span class="c3"></span></p><p class="c4"><span class="c1">In case of multiple &#39;related&#39; pairs, the list generated above can be extended using the same method as for our lone &#39;related&#39; pair.</span></p><p class="c2"><span class="c1"></span></p><p class="c4"><span class="c1">Delete the individuals with the lowest call rate in &#39;related&#39; pairs with a pihat &gt; 0.2:</span></p><p class="c4"><span class="c3">plink --bfile HapMap_3_r3_11 --remove 0.2_low_call_rate_pihat.txt --make-bed --out HapMap_3_r3_12</span></p><p class="c2"><span class="c1"></span></p><ol class="c19 lst-kix_wljc02q05kpj-0" start="7"><li class="c4 c11"><span class="c20">Population stratification </span></li></ol><p class="c4"><span class="c24">Population stratification -- this is the presence of multiple subpopulations (e.g., individuals with different ethnic backgrounds) in a study. Because allele frequencies can differ between subpopulations, population stratification can lead to false positive associations and/or mask true associations. An excellent example of this is the chopstick gene, where a SNP, due to population stratification, accounted for nearly half of the variance in the capacity to eat with chopsticks [</span><span class="c25 c24"><a class="c8" href="https://www.google.com/url?q=https://www.ncbi.nlm.nih.gov/pubmed/10673763&amp;sa=D&amp;ust=1608477671583000&amp;usg=AOvVaw3RPJfL650F4lVako9vhVHJ">5</a></span><span class="c1">].</span></p><p class="c4"><span class="c24">There are several methods to correct for population stratification [</span><span class="c24 c25"><a class="c8" href="https://www.google.com/url?q=https://www.nature.com/articles/nrg2813&amp;sa=D&amp;ust=1608477671583000&amp;usg=AOvVaw3c7o3XCpTjP-cWJodXA5u3">6</a></span><span class="c24">]. In this tutorial, we illustrate a method that is incorporated in PLINK: the multidimensional scaling (MDS) approach. This method calculates the genome&#8208;wide average proportion of alleles shared between any pair of individuals within the sample to generate quantitative indices (components) of the genetic variation for each individual. The individual component scores can be plotted to explore whether there are groups of individuals that are genetically more similar to each other than expected. For example, in a genetic study including subjects from Asia and Europe, MDS analysis would reveal that Asians are genetically more similar to each other than to Europeans. To investigate for which individuals the generated component scores deviate from the samples target population, plotting of the scores of the sample under investigation and a population of known ethnic structure (e.g., HapMap/1KG data) is helpful: this step is called anchoring. This enables the researcher to obtain ethnic information on their sample and to determine possible ethnic outliers. In this tutorial proposed to perform MDS on your own data anchored by data of the 1KG project (</span><span class="c25 c24"><a class="c8" href="https://www.google.com/url?q=http://www.1000genomes.org/&amp;sa=D&amp;ust=1608477671584000&amp;usg=AOvVaw27Fc7m3BhA_bJ7s6XtDDvG">http://www.1000genomes.org/</a></span><span class="c1">). </span></p><p class="c2"><span class="c1"></span></p><p class="c4"><span class="c1">Download 1000 Genomes data, containing genetic data of 629 individuals from different ethnic backgrounds.</span></p><p class="c4"><span class="c24">Due to the large file size and the lack of processing power of our computer, we are forced to skip this step. However, we can look at the results of this work taken from the article [</span><span class="c25 c24"><a class="c8" href="https://www.google.com/url?q=https://doi.org/10.1002/mpr.1608&amp;sa=D&amp;ust=1608477671584000&amp;usg=AOvVaw1mta8_cxLntXTlrOKthpP9">7</a></span><span class="c1">]. </span></p><p class="c2"><span class="c1"></span></p><p class="c4"><span style="overflow: hidden; display: inline-block; margin: 0.00px 0.00px; border: 0.00px solid #000000; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px); width: 288.78px; height: 271.79px;"><img alt="" src="images/image18.png" style="width: 292.96px; height: 271.79px; margin-left: 0.00px; margin-top: 0.00px; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px);" title=""></span></p><p class="c4"><span class="c9">Fig. 7. Multidimensional scaling (MDS) plot of 1KG against the CEU of the HapMap data. From article [7]</span></p><p class="c2"><span class="c9"></span></p><p class="c2"><span class="c9"></span></p><p class="c4"><span class="c24">This plot shows a multidimensional scaling (MDS) plot of 1KG against the CEU of the HapMap data. The black crosses (+ = &ldquo;OWN&rdquo;) in the upper left part represent the first two MDS components of the individuals in the HapMap sample (the coloured symbols represent the 1KG data (</span><span style="overflow: hidden; display: inline-block; margin: 0.00px 0.00px; border: 0.00px solid #000000; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px); width: 10.00px; height: 10.00px;"><img alt="An external file that holds a picture, illustration, etc.
Object name is MPR-27-e1608-g005.jpg" src="images/image13.jpg" style="width: 10.00px; height: 10.00px; margin-left: 0.00px; margin-top: 0.00px; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px);" title=""></span><span class="c24">&nbsp;= European; </span><span style="overflow: hidden; display: inline-block; margin: 0.00px 0.00px; border: 0.00px solid #000000; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px); width: 11.00px; height: 11.00px;"><img alt="An external file that holds a picture, illustration, etc.
Object name is MPR-27-e1608-g006.jpg" src="images/image7.jpg" style="width: 11.00px; height: 11.00px; margin-left: 0.00px; margin-top: 0.00px; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px);" title=""></span><span class="c24">&nbsp;= African; </span><span style="overflow: hidden; display: inline-block; margin: 0.00px 0.00px; border: 0.00px solid #000000; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px); width: 9.00px; height: 10.00px;"><img alt="An external file that holds a picture, illustration, etc.
Object name is MPR-27-e1608-g007.jpg" src="images/image2.jpg" style="width: 9.00px; height: 10.00px; margin-left: 0.00px; margin-top: 0.00px; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px);" title=""></span><span class="c24">&nbsp;= Ad Mixed American; </span><span style="overflow: hidden; display: inline-block; margin: 0.00px 0.00px; border: 0.00px solid #000000; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px); width: 10.00px; height: 10.00px;"><img alt="An external file that holds a picture, illustration, etc.
Object name is MPR-27-e1608-g008.jpg" src="images/image19.jpg" style="width: 10.00px; height: 10.00px; margin-left: 0.00px; margin-top: 0.00px; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px);" title=""></span><span class="c24">&nbsp;= Asian). The MDS components representing the European samples (</span><span style="overflow: hidden; display: inline-block; margin: 0.00px 0.00px; border: 0.00px solid #000000; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px); width: 10.00px; height: 10.00px;"><img alt="An external file that holds a picture, illustration, etc.
Object name is MPR-27-e1608-g010.jpg" src="images/image11.jpg" style="width: 10.00px; height: 10.00px; margin-left: 0.00px; margin-top: 0.00px; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px);" title=""></span><span class="c24">) are located in the upper left, the African samples (</span><span style="overflow: hidden; display: inline-block; margin: 0.00px 0.00px; border: 0.00px solid #000000; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px); width: 11.00px; height: 11.00px;"><img alt="An external file that holds a picture, illustration, etc.
Object name is MPR-27-e1608-g009.jpg" src="images/image7.jpg" style="width: 11.00px; height: 11.00px; margin-left: 0.00px; margin-top: 0.00px; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px);" title=""></span><span class="c24">) are located in the upper right, the Ad Mixed American samples (</span><span style="overflow: hidden; display: inline-block; margin: 0.00px 0.00px; border: 0.00px solid #000000; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px); width: 9.00px; height: 10.00px;"><img alt="An external file that holds a picture, illustration, etc.
Object name is MPR-27-e1608-g011.jpg" src="images/image2.jpg" style="width: 9.00px; height: 10.00px; margin-left: 0.00px; margin-top: 0.00px; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px);" title=""></span><span class="c24">) are located near the intersection point of the dashed lines, the Asian components (</span><span style="overflow: hidden; display: inline-block; margin: 0.00px 0.00px; border: 0.00px solid #000000; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px); width: 10.00px; height: 10.00px;"><img alt="An external file that holds a picture, illustration, etc.
Object name is MPR-27-e1608-g012.jpg" src="images/image14.jpg" style="width: 10.00px; height: 10.00px; margin-left: 0.00px; margin-top: 0.00px; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px);" title=""></span><span class="c1">) are located in the lower left part.</span></p><p class="c2"><span class="c1"></span></p><p class="c4"><span class="c1">This figure illustrates an example of such an analysis. Individuals who are outliers based on the MDS analysis should be removed from further analyses. After the exclusion of these individuals, a new MDS analysis must be conducted, and its main components need to be used as covariates in the association tests in order to correct for any remaining population stratification within the population. How many components need to be included depends on the population structure and the sample size, but the inclusion of up to 10 components is generally accepted within the psychiatric genetics community.</span></p><p class="c2"><span class="c1"></span></p><p class="c4"><span class="c1">We should perform an MDS only on data without ethnic outliers. But as we said above, we cannot analyze the stratification of the population, besides, we do not see significant outliers in our graph. Therefore, below we will use the data from the previous step HapMap_3_r3_12. </span></p><p class="c2"><span class="c1"></span></p><p class="c4"><span class="c3">plink --bfile HapMap_3_r3_13 --extract indepSNP.prune.in --genome --out HapMap_3_r3_13</span></p><p class="c4"><span class="c3">plink --bfile HapMap_3_r3_13 --read-genome HapMap_3_r3_13.genome --cluster --mds-plot 10 --out HapMap_3_r3_13_mds</span></p><p class="c2"><span class="c1"></span></p><p class="c4"><span class="c1">Change the format of the .mds file into a plink covariate file:</span></p><p class="c4"><span class="c3">awk &#39;{print$1, $2, $4, $5, $6, $7, $8, $9, $10, $11, $12, $13}&#39; HapMap_3_r3_13_mds.mds &gt; covar_mds.txt</span></p><p class="c2"><span class="c1"></span></p><p class="c4"><span class="c1">The values in covar_mds.txt will be used as covariates, to adjust for remaining population stratification, onward, where we will perform genome-wide association analysis.</span></p><p class="c2"><span class="c1"></span></p><p class="c4"><span class="c20">Statistical tests of associations</span></p><p class="c4"><span class="c24">Now our data is ready for associative tests. Within PLINK, the association between SNPs and a binary outcome can be tested with the options </span><span class="c13">&#8208;&#8208;assoc</span><span class="c24">&nbsp;or </span><span class="c13">&#8208;&#8208;logistic</span><span class="c24">. The </span><span class="c13">&#8208;&#8208;assoc</span><span class="c24">&nbsp;option in PLINK performs a X</span><span class="c24 c33">2</span><span class="c24">&nbsp;test of association that does not allow the inclusion of covariates. With the </span><span class="c13">&#8208;&#8208;logistic</span><span class="c1">&nbsp;option, a logistic regression analysis will be performed which allows the inclusion of covariates. </span></p><ol class="c19 lst-kix_kg9nuuktziog-0 start" start="1"><li class="c4 c11"><span class="c23">--assoc and --logistic &nbsp;tests</span></li></ol><p class="c4"><span class="c3">plink --bfile HapMap_3_r3_13 --assoc --out assoc_results</span></p><p class="c2"><span class="c1"></span></p><p class="c4"><span class="c3">plink --bfile HapMap_3_r3_13 --covar covar_mds.txt --logistic --hide-covar --out logistic_results</span></p><p class="c2"><span class="c1"></span></p><p class="c4"><span class="c1">Remove NA values, those might give problems generating plots in later steps:</span></p><p class="c4"><span class="c3">awk &#39;!/&#39;NA&#39;/&#39; logistic_results.assoc.logistic &gt; logistic_results.assoc_2.logistic</span></p><p class="c2"><span class="c1"></span></p><ol class="c19 lst-kix_kg9nuuktziog-0" start="2"><li class="c4 c11"><span class="c20">Multiple testing</span></li></ol><p class="c4"><span class="c1">Modern genotyping arrays can genotype up to 4 million markers concurrently, which generates a large number of tests, and thus, a considerable multiple testing burden. Three widely applied alternatives for determining genome&#8208;wide significance are the use of Bonferroni correction, Benjamini&ndash;Hochberg false discovery rate (FDR), and permutation testing. The Bonferroni correction is often too conservative and leads to an increase in the proportion of false negative findings because many SNPs are correlated, due to Linkage Disequilibrium (LD) and are thus by definition not independent. &nbsp;</span></p><p class="c4"><span class="c24">FDR controls the expected proportion of false positives among all signals with an FDR value below a fixed threshold, assuming that SNPs are independent [</span><span class="c25 c24"><a class="c8" href="https://www.google.com/url?q=https://www.bibsonomy.org/bibtex/b38b0e6655978ad8c7d8455b175c2cbf&amp;sa=D&amp;ust=1608477671589000&amp;usg=AOvVaw3EY19eIb4zN2ktEfL_UOE_">8</a></span><span class="c24">]. This method is less conservative than Bonferroni correction. To easily apply Bonferroni and FDR correction, PLINK offers the option </span><span class="c13">&#8208;&#8208;adjust</span><span class="c1">&nbsp;that generates output in which the unadjusted p-value is displayed, along with p values corrected with various multiple testing correction methods.</span></p><p class="c4"><span class="c24">Finally, permutation methods can be used to deal with the multiple testing burden. To calculate permutation&#8208;based p values, the outcome measure labels are randomly permuted multiple (e.g., 1,000&ndash;1,000,000) times which effectively removes any true association between the outcome measure and the genotype. For all permuted data sets, statistical tests are then performed. This provides the empirical distribution of the test&#8208;statistic and the p values under the null hypothesis of no association. The original test statistic or p-value obtained from the observed data is subsequently compared to the empirical distribution of p values to determine an empirically adjusted p-value. To use this method, the two PLINK options</span><span class="c34">&nbsp;</span><span class="c13">&#8208;&#8208;assoc</span><span class="c24">&nbsp;and </span><span class="c13">&#8208;&#8208;mperm</span><span class="c24">&nbsp;can be combined to generate two p values: EMP1, the empirical p-value (uncorrected), and EMP2, the empirical p-value corrected for multiple testing. This procedure is computationally intensive, especially if many permutations are required, which is necessary to calculate very small p values accurately [</span><span class="c25 c24"><a class="c8" href="https://www.google.com/url?q=https://pubmed.ncbi.nlm.nih.gov/12596795/&amp;sa=D&amp;ust=1608477671590000&amp;usg=AOvVaw0woaBhZMsfp_z1TrtYhqyO">9</a></span><span class="c1">].</span></p><p class="c2"><span class="c1"></span></p><p class="c4"><span class="c23 c37 c17">Adjust</span></p><p class="c4"><span class="c3">plink --bfile HapMap_3_r3_13 -assoc --adjust --out adjusted_assoc_results</span></p><p class="c4"><span class="c1">This file gives a Bonferroni corrected p-value, along with FDR and others.</span></p><p class="c2"><span class="c1"></span></p><p class="c4"><span class="c23 c17">Permutation</span></p><p class="c4"><span class="c1">To reduce the computational time we only perform this test on a subset of the SNPs from chromosome 22. The EMP2 column provides for multiple testing corrected p-value.</span></p><p class="c4"><span class="c1">Generate subset of SNPs:</span></p><p class="c4"><span class="c3">awk &#39;{ if ($4 &gt;= 21595000 &amp;&amp; $4 &lt;= 21605000) print $2 }&#39; HapMap_3_r3_13.bim &gt; subset_snp_chr_22.txt</span></p><p class="c2"><span class="c3"></span></p><p class="c4"><span class="c1">Filter your bfile based on the subset of SNPs generated in the step above:</span></p><p class="c4"><span class="c3">plink --bfile HapMap_3_r3_13 --extract subset_snp_chr_22.txt --make-bed --out HapMap_subset_for_perm</span></p><p class="c2"><span class="c3"></span></p><p class="c4"><span class="c1">Perform 1000000 permutations:</span></p><p class="c4"><span class="c3">plink --bfile HapMap_subset_for_perm --assoc --mperm 1000000 --out subset_1M_perm_result</span></p><p class="c2"><span class="c1"></span></p><p class="c4"><span class="c1">Order your data, from lowest to highest p-value:</span></p><p class="c4"><span class="c3">sort -gk 4 subset_1M_perm_result.assoc.mperm &gt; sorted_subset.txt</span></p><p class="c2"><span class="c1"></span></p><p class="c4"><span class="c1">Check ordered permutation results:</span></p><p class="c4"><span class="c3">head sorted_subset.txt</span></p><p class="c2"><span class="c1"></span></p><ol class="c19 lst-kix_kg9nuuktziog-0" start="3"><li class="c4 c11"><span class="c20">Generate Manhattan and QQ plots</span></li></ol><p class="c2"><span class="c20"></span></p><p class="c4"><span class="c3">Rscript --no-save Manhattan_plot.R</span></p><p class="c2"><span class="c9"></span></p><a id="t.65aaff097b720e46597af9f380027fa30f2a5bd0"></a><a id="t.0"></a><table class="c30"><tbody><tr class="c38"><td class="c22" colspan="1" rowspan="1"><p class="c32"><span style="overflow: hidden; display: inline-block; margin: 0.00px 0.00px; border: 0.00px solid #000000; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px); width: 268.00px; height: 256.09px;"><img alt="" src="images/image4.png" style="width: 268.00px; height: 256.09px; margin-left: 0.00px; margin-top: 0.00px; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px);" title=""></span></p></td><td class="c22" colspan="1" rowspan="1"><p class="c32"><span style="overflow: hidden; display: inline-block; margin: 0.00px 0.00px; border: 0.00px solid #000000; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px); width: 250.50px; height: 245.19px;"><img alt="" src="images/image16.png" style="width: 250.50px; height: 245.19px; margin-left: 0.00px; margin-top: 0.00px; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px);" title=""></span></p></td></tr></tbody></table><p class="c2"><span class="c9"></span></p><p class="c4"><span class="c18">Fig. 8. Manhatten plots for two ways of analysis: assos ( X</span><span class="c18 c33">2</span><span class="c9">&nbsp;test of association) and logistic (including covariates). &nbsp;Genomic coordinates are displayed along the X-axis, with the negative logarithm of the association p-value for each SNP &nbsp;displayed on the Y-axis, meaning that each dot on the plot signifies a SNP. Because the strongest associations have the smallest p-values , their negative logarithms are the greatest. </span></p><p class="c2"><span class="c1"></span></p><p class="c2"><span class="c1"></span></p><p class="c4"><span class="c20">Step 2. Comparative analysis of articles on GWAS studies of patients with COVID-19</span></p><p class="c4"><span class="c24">We compared the search results for gene variants associated with severe COVID-19, obtained in articles [</span><span class="c25 c24"><a class="c8" href="https://www.google.com/url?q=https://doi.org/10.1038/s41586-020-2818-3&amp;sa=D&amp;ust=1608477671594000&amp;usg=AOvVaw1eqrBk_RXc3A9RhNHPDEnV">10</a></span><span class="c24">] and [</span><span class="c25 c24"><a class="c8" href="https://www.google.com/url?q=https://doi.org/10.1038/s41421-020-00231-4&amp;sa=D&amp;ust=1608477671594000&amp;usg=AOvVaw0lm8urGPjmglRfJB5Lbxed">11</a></span><span class="c1">]. The results of these works can be seen on Fig. 9 and Fig. 10. &nbsp; </span></p><p class="c4"><span style="overflow: hidden; display: inline-block; margin: 0.00px 0.00px; border: 0.00px solid #000000; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px); width: 395.50px; height: 250.34px;"><img alt="" src="images/image1.png" style="width: 395.50px; height: 278.68px; margin-left: 0.00px; margin-top: -28.34px; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px);" title=""></span></p><p class="c4"><span class="c24">&nbsp; </span><span class="c18">&nbsp;Fig. 9. A genome-wide study of associations of 3,199 hospitalized patients with COVID-19 and 897,488 controls. Genotyped data from COVID-19 Host Genetics Initiative (</span><span class="c6"><a class="c8" href="https://www.google.com/url?q=https://www.covid19hg.org/&amp;sa=D&amp;ust=1608477671595000&amp;usg=AOvVaw0I5lQRbAE9IAcqJ4PtixF4">https://www.covid19hg.org/</a></span><span class="c9">). From article [10]. &nbsp; </span></p><p class="c2"><span class="c20"></span></p><p class="c4"><span style="overflow: hidden; display: inline-block; margin: 0.00px 0.00px; border: 0.00px solid #000000; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px); width: 385.02px; height: 214.64px;"><img alt="" src="images/image20.png" style="width: 393.30px; height: 214.64px; margin-left: -8.28px; margin-top: 0.00px; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px);" title=""></span><span style="overflow: hidden; display: inline-block; margin: 0.00px 0.00px; border: 0.00px solid #000000; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px); width: 393.50px; height: 219.75px;"><img alt="" src="images/image6.png" style="width: 393.50px; height: 276.42px; margin-left: 0.00px; margin-top: -38.96px; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px);" title=""></span></p><p class="c4"><span class="c18">Fig. 10. Genome-wide study of associations of 284 hospitalized patients and 301 control individuals. Up: one-way association tests; down: associative tests based on gene interactions. Deep sequencing data from CNSA (China National Genebank Sequence Archive, </span><span class="c6"><a class="c8" href="https://www.google.com/url?q=https://db.cngb.org/cnsa/&amp;sa=D&amp;ust=1608477671596000&amp;usg=AOvVaw2EyoJ_JU1scC-U8CledveO">https://db.cngb.org/cnsa/</a></span><span class="c18">). From article [11]. </span><span class="c20">&nbsp; </span></p><p class="c2"><span class="c20"></span></p><p class="c12"><span class="c1">The first work revealed an association of an extended region of the third chromosome, which is most likely inherited from the Neanderthals. Also, an association was found with a small region of chromosome 9 (including 1 gene), but later analysis did not confirm their relationship.</span></p><p class="c12"><span class="c1">In the second work, GWAS was carried out both at the SNP level and at the level of alleles of individual genes. Two different groups were selected as control groups, depending on which the detected genes and SNPs significantly changed.</span></p><p class="c12"><span class="c1">We believe that in the studies under consideration, the results differ for the following reasons:</span></p><p class="c12 c35"><span class="c1">1. Different sample sizes (3199/897488 and 284/301)</span></p><p class="c12 c27"><span class="c1">2. Different methods of obtaining SNP (genotyping using microarrays and deep sequencing)</span></p><p class="c12 c27"><span class="c1">3. Different composition of samples (in the first case, different ethnic groups, respectively, were applied stratification corrections, in the second - people of the same race).</span></p><p class="c7"><span class="c1"></span></p><p class="c7"><span class="c1"></span></p><p class="c4"><span class="c20">Conclusion</span></p><p class="c4"><span class="c24">This is where our introduction to plink and GWAS ends. We looked at the basic theory underlying GWAS analysis, as well as completed the main steps of this analysis. In addition, we analyzed the results obtained in various GWAS reseaches. We hope this knowledge will be </span><span class="c24">useful to us in the future.</span></p><p class="c2"><span class="c20"></span></p><p class="c4"><span class="c23">References</span></p><ol class="c19 lst-kix_bano2ezcy3fm-0 start" start="1"><li class="c4 c11"><span class="c10"><a class="c8" href="https://www.google.com/url?q=https://doi.org/10.1017/S0033291714002451&amp;sa=D&amp;ust=1608477671597000&amp;usg=AOvVaw0UvXPLq_GjEpwuXzPLY538">Purcell, S., Neale, B., Todd-Brown, K., Thomas, L., Ferreira, M. A., Bender, D., ... &amp; Sham, P. C. (2007). PLINK: a tool set for whole-genome association and population-based linkage analyses. </a></span><span class="c10 c17"><a class="c8" href="https://www.google.com/url?q=https://doi.org/10.1017/S0033291714002451&amp;sa=D&amp;ust=1608477671598000&amp;usg=AOvVaw2pvfk7EINRiA6c8t_GnX_c">The American journal of human genetics</a></span><span class="c10"><a class="c8" href="https://www.google.com/url?q=https://doi.org/10.1017/S0033291714002451&amp;sa=D&amp;ust=1608477671598000&amp;usg=AOvVaw2pvfk7EINRiA6c8t_GnX_c">, </a></span><span class="c10 c17"><a class="c8" href="https://www.google.com/url?q=https://doi.org/10.1017/S0033291714002451&amp;sa=D&amp;ust=1608477671598000&amp;usg=AOvVaw2pvfk7EINRiA6c8t_GnX_c">81</a></span><span class="c10"><a class="c8" href="https://www.google.com/url?q=https://doi.org/10.1017/S0033291714002451&amp;sa=D&amp;ust=1608477671598000&amp;usg=AOvVaw2pvfk7EINRiA6c8t_GnX_c">(3), 559-575</a></span><span class="c1 c14">.</span></li><li class="c4 c11"><span class="c10"><a class="c8" href="https://www.google.com/url?q=http://dx.doi.org/10.1038/nature02168&amp;sa=D&amp;ust=1608477671599000&amp;usg=AOvVaw1xvvLLf9oKVBHnIDWDJju1">Gibbs, R. A., Belmont, J. W., Hardenbol, P., Willis, T. D., Yu, F. L., Yang, H. M., ... &amp; Tam, P. K. H. (2003). The international HapMap project.</a></span></li><li class="c4 c11"><span class="c10"><a class="c8" href="https://www.google.com/url?q=https://www.thelancet.com/journals/lanonc/article/PIIS1470-2045(10)70042-5/fulltext&amp;sa=D&amp;ust=1608477671599000&amp;usg=AOvVaw2dRm-xblmz_lLtEz9vlThU">Li, Y., Sheu, C. C., Ye, Y., De Andrade, M., Wang, L., Chang, S. C., ... &amp; Cunningham, J. M. (2010). Genetic variants and risk of lung cancer in never smokers: a genome-wide association study. </a></span><span class="c10 c17"><a class="c8" href="https://www.google.com/url?q=https://www.thelancet.com/journals/lanonc/article/PIIS1470-2045(10)70042-5/fulltext&amp;sa=D&amp;ust=1608477671599000&amp;usg=AOvVaw2dRm-xblmz_lLtEz9vlThU">The lancet oncology</a></span><span class="c10"><a class="c8" href="https://www.google.com/url?q=https://www.thelancet.com/journals/lanonc/article/PIIS1470-2045(10)70042-5/fulltext&amp;sa=D&amp;ust=1608477671600000&amp;usg=AOvVaw2zHPYLi_j6SSuoqGkY6Dm7">, </a></span><span class="c10 c17"><a class="c8" href="https://www.google.com/url?q=https://www.thelancet.com/journals/lanonc/article/PIIS1470-2045(10)70042-5/fulltext&amp;sa=D&amp;ust=1608477671600000&amp;usg=AOvVaw2zHPYLi_j6SSuoqGkY6Dm7">11</a></span><span class="c10"><a class="c8" href="https://www.google.com/url?q=https://www.thelancet.com/journals/lanonc/article/PIIS1470-2045(10)70042-5/fulltext&amp;sa=D&amp;ust=1608477671601000&amp;usg=AOvVaw2YIWPuLSMfZZySVjc_jigr">(4), 321-330</a></span><span class="c1 c14">.</span></li><li class="c39 c11 c14"><span class="c10"><a class="c8" href="https://www.google.com/url?q=https://pubmed.ncbi.nlm.nih.gov/21085122/&amp;sa=D&amp;ust=1608477671601000&amp;usg=AOvVaw0Vvc7Hg9Elw-4BBTFZd067">Anderson, C. A. , Pettersson, F. H. , Clarke, G. M. , Cardon, L. R. , Morris, A. P. , &amp; Zondervan, K. T. (2010). Data quality control in genetic case&#8208;control association studies. Nature Protocols, 5(9), 1564&ndash;1573. 10.1038/nprot.2010.116</a></span></li><li class="c11 c14 c39"><span class="c10"><a class="c8" href="https://www.google.com/url?q=https://www.ncbi.nlm.nih.gov/pubmed/10673763&amp;sa=D&amp;ust=1608477671601000&amp;usg=AOvVaw1fgrQFwpzotXTWWu_DI5Yb">Hamer D, Sirota L. Beware the chopsticks gene. Mol Psychiatry. 2000 Jan;5(1):11-3. doi: 10.1038/sj.mp.4000662. PMID: 10673763.</a></span></li><li class="c4 c11"><span class="c10"><a class="c8" href="https://www.google.com/url?q=https://www.nature.com/articles/nrg2813&amp;sa=D&amp;ust=1608477671602000&amp;usg=AOvVaw0kUiaUTSDz3SeecQ__zwn9">Price, A. L., Zaitlen, N. A., Reich, D., &amp; Patterson, N. (2010). New approaches to population stratification in genome-wide association studies. </a></span><span class="c10 c17"><a class="c8" href="https://www.google.com/url?q=https://www.nature.com/articles/nrg2813&amp;sa=D&amp;ust=1608477671602000&amp;usg=AOvVaw0kUiaUTSDz3SeecQ__zwn9">Nature Reviews Genetics</a></span><span class="c10"><a class="c8" href="https://www.google.com/url?q=https://www.nature.com/articles/nrg2813&amp;sa=D&amp;ust=1608477671602000&amp;usg=AOvVaw0kUiaUTSDz3SeecQ__zwn9">, </a></span><span class="c10 c17"><a class="c8" href="https://www.google.com/url?q=https://www.nature.com/articles/nrg2813&amp;sa=D&amp;ust=1608477671603000&amp;usg=AOvVaw0-Hz1eVMnLpd_J1bxEAZxw">11</a></span><span class="c10"><a class="c8" href="https://www.google.com/url?q=https://www.nature.com/articles/nrg2813&amp;sa=D&amp;ust=1608477671603000&amp;usg=AOvVaw0-Hz1eVMnLpd_J1bxEAZxw">(7), 459-463</a></span></li><li class="c4 c11"><span class="c10"><a class="c8" href="https://www.google.com/url?q=https://doi.org/10.1002/mpr.1608&amp;sa=D&amp;ust=1608477671603000&amp;usg=AOvVaw2SVFxIxoRc4Xg_RtV489fY">Andries T. Marees, Hilde de Kluiver, et al. (2018). A tutorial on conducting genome&#8208;wide association studies: Quality control and statistical analysis. </a></span><span class="c10 c17"><a class="c8" href="https://www.google.com/url?q=https://doi.org/10.1002/mpr.1608&amp;sa=D&amp;ust=1608477671604000&amp;usg=AOvVaw3YYvFvEN7I0JtgaLNG8kRj">International Journal of Methods in Psychiatric Research, 2018 Jun; 27(2): e1608.</a></span></li><li class="c4 c11"><span class="c10"><a class="c8" href="https://www.google.com/url?q=https://www.bibsonomy.org/bibtex/b38b0e6655978ad8c7d8455b175c2cbf&amp;sa=D&amp;ust=1608477671604000&amp;usg=AOvVaw0z1gVor0RO2qcLlGPg8iXL">Benjamini, Y., </a></span><span class="c10"><a class="c8" href="https://www.google.com/url?q=https://www.bibsonomy.org/bibtex/b38b0e6655978ad8c7d8455b175c2cbf&amp;sa=D&amp;ust=1608477671605000&amp;usg=AOvVaw3M7v80a6xk7gzNZ3vwp1y7">&amp; Hochberg</a></span><span class="c10"><a class="c8" href="https://www.google.com/url?q=https://www.bibsonomy.org/bibtex/b38b0e6655978ad8c7d8455b175c2cbf&amp;sa=D&amp;ust=1608477671605000&amp;usg=AOvVaw3M7v80a6xk7gzNZ3vwp1y7">, Y. (1995). Controlling the false discovery rate: a practical and powerful approach to multiple testing. </a></span><span class="c10 c17"><a class="c8" href="https://www.google.com/url?q=https://www.bibsonomy.org/bibtex/b38b0e6655978ad8c7d8455b175c2cbf&amp;sa=D&amp;ust=1608477671605000&amp;usg=AOvVaw3M7v80a6xk7gzNZ3vwp1y7">Journal of the Royal statistical society: series B (Methodological)</a></span><span class="c10"><a class="c8" href="https://www.google.com/url?q=https://www.bibsonomy.org/bibtex/b38b0e6655978ad8c7d8455b175c2cbf&amp;sa=D&amp;ust=1608477671605000&amp;usg=AOvVaw3M7v80a6xk7gzNZ3vwp1y7">, </a></span><span class="c10 c17"><a class="c8" href="https://www.google.com/url?q=https://www.bibsonomy.org/bibtex/b38b0e6655978ad8c7d8455b175c2cbf&amp;sa=D&amp;ust=1608477671606000&amp;usg=AOvVaw1Qv70MwhQEaeP5Oe4PJ53E">57</a></span><span class="c10"><a class="c8" href="https://www.google.com/url?q=https://www.bibsonomy.org/bibtex/b38b0e6655978ad8c7d8455b175c2cbf&amp;sa=D&amp;ust=1608477671606000&amp;usg=AOvVaw1Qv70MwhQEaeP5Oe4PJ53E">(1), 289-300.</a></span></li><li class="c4 c11"><span class="c10"><a class="c8" href="https://www.google.com/url?q=https://pubmed.ncbi.nlm.nih.gov/12596795/&amp;sa=D&amp;ust=1608477671607000&amp;usg=AOvVaw2QwnH-MujDp2mygAbRZYrv">North, B. V., Curtis, D., &amp; Sham, P. C. (2003). A note on the calculation of empirical P values from Monte Carlo procedures. </a></span><span class="c10 c17"><a class="c8" href="https://www.google.com/url?q=https://pubmed.ncbi.nlm.nih.gov/12596795/&amp;sa=D&amp;ust=1608477671607000&amp;usg=AOvVaw2QwnH-MujDp2mygAbRZYrv">The American Journal of Human Genetics</a></span><span class="c10"><a class="c8" href="https://www.google.com/url?q=https://pubmed.ncbi.nlm.nih.gov/12596795/&amp;sa=D&amp;ust=1608477671607000&amp;usg=AOvVaw2QwnH-MujDp2mygAbRZYrv">, </a></span><span class="c10 c17"><a class="c8" href="https://www.google.com/url?q=https://pubmed.ncbi.nlm.nih.gov/12596795/&amp;sa=D&amp;ust=1608477671607000&amp;usg=AOvVaw2QwnH-MujDp2mygAbRZYrv">72</a></span><span class="c10"><a class="c8" href="https://www.google.com/url?q=https://pubmed.ncbi.nlm.nih.gov/12596795/&amp;sa=D&amp;ust=1608477671608000&amp;usg=AOvVaw3aFIRf2TqMByinQrVTr1gu">(2), 498-499.</a></span></li><li class="c4 c11"><span class="c10"><a class="c8" href="https://www.google.com/url?q=https://doi.org/10.1038/s41586-020-2818-3&amp;sa=D&amp;ust=1608477671608000&amp;usg=AOvVaw1FAB4wcEVPhIla6dxOWbQk">Zeberg, H., P&auml;&auml;bo, S. The major genetic risk factor for severe COVID-19 is inherited from Neanderthals. </a></span><span class="c10 c17"><a class="c8" href="https://www.google.com/url?q=https://doi.org/10.1038/s41586-020-2818-3&amp;sa=D&amp;ust=1608477671608000&amp;usg=AOvVaw1FAB4wcEVPhIla6dxOWbQk">Nature</a></span><span class="c10"><a class="c8" href="https://www.google.com/url?q=https://doi.org/10.1038/s41586-020-2818-3&amp;sa=D&amp;ust=1608477671608000&amp;usg=AOvVaw1FAB4wcEVPhIla6dxOWbQk">&nbsp;587, 610&ndash;612 (2020). https://doi.org/10.1038/s41586-020-2818-3</a></span></li><li class="c4 c11"><span class="c24"><a class="c8" href="https://www.google.com/url?q=https://doi.org/10.1038/s41421-020-00231-4&amp;sa=D&amp;ust=1608477671609000&amp;usg=AOvVaw0ZoGGdzy4rxY-IS0UgV98Z">Wang, F., Huang, S., Gao, R. et al. Initial whole-genome sequencing and analysis of the host genetic contribution to COVID-19 severity and susceptibility. Cell Discov 6, 83 (2020). https://doi.org/10.1038/s41421-020-00231-4</a></span></li></ol><div><p class="c5 c21"><span class="c36"></span></p></div></body></html>